<article xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"><?properties open_access>
<front>
<journal-meta>
<journal-id journal-id-type="nlm-ta">PLoS One</journal-id>
<journal-id journal-id-type="publisher-id">plos</journal-id>
<journal-id journal-id-type="pmc">plosone</journal-id>
<journal-title-group>
<journal-title>PLoS ONE</journal-title>
</journal-title-group>
<issn pub-type="epub">1932-6203</issn>
<publisher>
<publisher-name>Public Library of Science</publisher-name>
<publisher-loc>San Francisco, USA</publisher-loc>
</publisher>
</journal-meta>
<article-meta>
<article-id pub-id-type="pmid">21738781</article-id>
<article-id pub-id-type="pmc">3125343</article-id>
<article-id pub-id-type="publisher-id">PONE-D-11-07325</article-id>
<article-id pub-id-type="doi">10.1371/journal.pone.0021729</article-id>
<article-categories>
<subj-group subj-group-type="heading">
<subject>Research Article</subject>
</subj-group>
<subj-group subj-group-type="Discipline-v2">
<subject>Biology</subject>
<subj-group>
<subject>Biochemistry</subject>
<subj-group>
<subject>Proteins</subject>
<subj-group>
<subject>Protein Synthesis</subject>
</subj-group>
</subj-group>
</subj-group>
<subj-group>
<subject>Computational Biology</subject>
<subj-group>
<subject>Signaling Networks</subject>
</subj-group>
</subj-group>
<subj-group>
<subject>Molecular Cell Biology</subject>
<subj-group>
<subject>Signal Transduction</subject>
<subj-group>
<subject>Signaling in Cellular Processes</subject>
<subj-group>
<subject>Tor Signaling</subject>
</subj-group>
</subj-group>
<subj-group>
<subject>Mechanisms of Signal Transduction</subject>
</subj-group>
</subj-group>
<subj-group>
<subject>Cell Growth</subject>
</subj-group>
</subj-group>
<subj-group>
<subject>Proteomics</subject>
</subj-group>
</subj-group>
</article-categories>
<title-group>
<article-title>Phosphoproteomic Profiling of <italic>In Vivo</italic> Signaling in Liver by the Mammalian Target of Rapamycin Complex 1 (mTORC1)</article-title>
<alt-title alt-title-type="running-head">Phosphoproteomic Profiling of mTORC1 Signaling</alt-title>
</title-group>
<contrib-group>
<contrib contrib-type="author">
<name>
<surname>Demirkan</surname>
<given-names>Gokhan</given-names>
</name>
<xref ref-type="aff" rid="aff1">
<sup>1</sup>
</xref>
<xref ref-type="aff" rid="aff2">
<sup>2</sup>
</xref>
</contrib>
<contrib contrib-type="author">
<name>
<surname>Yu</surname>
<given-names>Kebing</given-names>
</name>
<xref ref-type="aff" rid="aff2">
<sup>2</sup>
</xref>
<xref ref-type="aff" rid="aff3">
<sup>3</sup>
</xref>
</contrib>
<contrib contrib-type="author">
<name>
<surname>Boylan</surname>
<given-names>Joan M.</given-names>
</name>
<xref ref-type="aff" rid="aff1">
<sup>1</sup>
</xref>
</contrib>
<contrib contrib-type="author">
<name>
<surname>Salomon</surname>
<given-names>Arthur R.</given-names>
</name>
<xref ref-type="aff" rid="aff2">
<sup>2</sup>
</xref>
<xref ref-type="aff" rid="aff3">
<sup>3</sup>
</xref>
</contrib>
<contrib contrib-type="author">
<name>
<surname>Gruppuso</surname>
<given-names>Philip A.</given-names>
</name>
<xref ref-type="aff" rid="aff1">
<sup>1</sup>
</xref>
<xref ref-type="aff" rid="aff2">
<sup>2</sup>
</xref>
<xref ref-type="corresp" rid="cor1">
<sup>*</sup>
</xref>
</contrib>
</contrib-group>
<aff id="aff1">
<label>1</label>
<addr-line>Department of Pediatrics, Brown University and Rhode Island Hospital, Providence, Rhode Island, United States of America</addr-line>
</aff>
<aff id="aff2">
<label>2</label>
<addr-line>Department of Molecular Biology, Cell Biology, and Biochemistry, Brown University, Providence, Rhode Island, United States of America</addr-line>
</aff>
<aff id="aff3">
<label>3</label>
<addr-line>Department of Chemistry, Brown University, Providence, Rhode Island, United States of America</addr-line>
</aff>
<contrib-group>
<contrib contrib-type="editor">
<name>
<surname>Polymenis</surname>
<given-names>Michael</given-names>
</name>
<role>Editor</role>
<xref ref-type="aff" rid="edit1"></xref>
</contrib>
</contrib-group>
<aff id="edit1">Texas A&amp;M University, United States of America</aff>
<author-notes>
<corresp id="cor1">* E-mail: <email>Philip_Gruppuso@brown.edu</email>
</corresp>
<fn fn-type="con">
<p>Conceived and designed the experiments: GD KY JMB ARS PAG. Performed the experiments: GD KY ARS. Analyzed the data: GD KY ARS PG. Wrote the paper: GD ARS PAG.</p>
</fn>
</author-notes>
<pub-date pub-type="collection">
<year>2011</year>
</pub-date>
<pub-date pub-type="epub">
<day>28</day>
<month>6</month>
<year>2011</year>
</pub-date>
<volume>6</volume>
<issue>6</issue>
<elocation-id>e21729</elocation-id>
<history>
<date date-type="received">
<day>21</day>
<month>4</month>
<year>2011</year>
</date>
<date date-type="accepted">
<day>7</day>
<month>6</month>
<year>2011</year>
</date>
</history>
<permissions>
<copyright-statement>Demirkan et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</copyright-statement>
<copyright-year>2011</copyright-year>
</permissions>
<abstract>
<sec>
<title>Background</title>
<p>Our understanding of signal transduction networks in the physiological context of an organism remains limited, partly due to the technical challenge of identifying serine/threonine phosphorylated peptides from complex tissue samples. In the present study, we focused on signaling through the mammalian target of rapamycin (mTOR) complex 1 (mTORC1), which is at the center of a nutrient- and growth factor-responsive cell signaling network. Though studied extensively, the mechanisms involved in many mTORC1 biological functions remain poorly understood.</p>
</sec>
<sec>
<title>Methodology/Principal Findings</title>
<p>We developed a phosphoproteomic strategy to purify, enrich and identify phosphopeptides from rat liver homogenates. Using the anticancer drug rapamycin, the only known target of which is mTORC1, we characterized signaling in liver from rats in which the complex was maximally activated by refeeding following 48 hr of starvation. Using protein and peptide fractionation methods, TiO<sub>2</sub> affinity purification of phosphopeptides and mass spectrometry, we reproducibly identified and quantified over four thousand phosphopeptides. Along with 5 known rapamycin-sensitive phosphorylation events, we identified 62 new rapamycin-responsive candidate phosphorylation sites. Among these were PRAS40, gephyrin, and AMP kinase 2. We observed similar proportions of increased and reduced phosphorylation in response to rapamycin. Gene ontology analysis revealed over-representation of mTOR pathway components among rapamycin-sensitive phosphopeptide candidates.</p>
</sec>
<sec>
<title>Conclusions/Significance</title>
<p>In addition to identifying potential new mTORC1-mediated phosphorylation events, and providing information relevant to the biology of this signaling network, our experimental and analytical approaches indicate the feasibility of large-scale phosphoproteomic profiling of tissue samples to study physiological signaling events <italic>in vivo</italic>.</p>
</sec>
</abstract>
<counts>
<page-count count="13"></page-count>
</counts>
</article-meta>
</front>
<body>
<sec id="s1">
<title>Introduction</title>
<p>The mammalian target of rapamycin (mTOR) is a well conserved serine/threonine kinase that plays a key physiological role in the control of cell growth <xref ref-type="bibr" rid="pone.0021729-Wullschleger1">[1]</xref>. mTOR is a component of two distinct multiprotein complexes <xref ref-type="bibr" rid="pone.0021729-Alessi1">[2]</xref>&ndash;<xref ref-type="bibr" rid="pone.0021729-Guertin1">[4]</xref>. mTOR complex 1 (mTORC1) regulates temporal control of cell growth while mTOR complex 2 (mTORC2) regulates the organization of the actin cytoskeleton. mTORC1 signaling is sensitive to rapamycin, a macrolide antibiotic and anti-cancer drug, whereas mTORC2 mediates rapamycin-insensitive signaling <xref ref-type="bibr" rid="pone.0021729-Abraham1">[5]</xref>. mTORC1 signaling is stimulated by nutrients, growth factors, and high levels of cellular energy. In addition to the inhibitory effect of rapamycin, mTORC1 signaling is downregulated by environmental stressors, such as hypoxia and low cellular energy levels. Activation of mTORC1 leads to increased ribosome biogenesis, translation and nutrient transport, and to repression of autophagy and stress-induced transcription <xref ref-type="bibr" rid="pone.0021729-Wullschleger1">[1]</xref>, <xref ref-type="bibr" rid="pone.0021729-Hay1">[6]</xref>.</p>
<p>The constituents of mTORC1 include mTOR, raptor, mLST8/G-protein β-subunit like protein (GβL), Proline-Rich Akt Substrate of 40 kDa (PRAS40), and Deptor <xref ref-type="bibr" rid="pone.0021729-Foster1">[7]</xref>. This signaling complex exerts at least some of its biological effects by phosphorylating p70 ribosomal protein S6 kinase (S6K) at a single site, and eukaryotic initiation factor 4E (eIF4E) binding protein 1 (4E-BP1) at multiple sites. mTORC1 phosphorylation of critical residues involved in the activation of S6K1 (Thr389) and 4E-BP1 (Thr37 and Thr46) are generally sensitive to the inhibitory effects of rapamycin <xref ref-type="bibr" rid="pone.0021729-Hidalgo1">[8]</xref>, <xref ref-type="bibr" rid="pone.0021729-Loewith1">[9]</xref>. Activated S6K1 functions <italic>in vivo</italic> to phosphorylate the 40S ribosomal protein S6, the biological significance of which is uncertain <xref ref-type="bibr" rid="pone.0021729-Wool1">[10]</xref>. The phosphorylation of 4E-BP1 promotes its dissociation from eIF4E, thus activating 5&prime;-cap-dependent mRNA translation, a process that accounts for the majority of total cellular translation <xref ref-type="bibr" rid="pone.0021729-Gingras1">[11]</xref>&ndash;<xref ref-type="bibr" rid="pone.0021729-Nojima1">[13]</xref>. Though much research has been devoted to understanding the mTORC1 pathway, the mechanisms underlying many of its biological functions remain poorly understood. It is likely that direct and indirect substrates of mTORC1 remain unidentified <xref ref-type="bibr" rid="pone.0021729-Wullschleger1">[1]</xref>, <xref ref-type="bibr" rid="pone.0021729-Dunlop1">[3]</xref>, <xref ref-type="bibr" rid="pone.0021729-Ma1">[12]</xref>&ndash;<xref ref-type="bibr" rid="pone.0021729-Polak1">[16]</xref>.</p>
<p>In the present study, we have utilized a well characterized model of mTORC1 activation <italic>in vivo</italic>, fasting followed by refeeding in the laboratory rat <xref ref-type="bibr" rid="pone.0021729-Anand1">[17]</xref>, <xref ref-type="bibr" rid="pone.0021729-Anand2">[18]</xref>. During a period of food deprivation lasting 48 hours, liver mass and protein content decrease by approximately one quarter and one third, respectively. Within one hour of refeeding, marked mTORC1 activation occurs. Rapamycin injection prior to refeeding prevents this activation <xref ref-type="bibr" rid="pone.0021729-Anand2">[18]</xref>. While several groups have undertaken phosphoproteomic profiling of liver tissue <xref ref-type="bibr" rid="pone.0021729-Jin1">[19]</xref>&ndash;<xref ref-type="bibr" rid="pone.0021729-Han2">[23]</xref>, none have used this approach to investigate rapamycin-sensitive phosphorylation events <italic>in vivo</italic>.</p>
<p>Recent advances in the technology of mass spectrometry (MS) permit the wide-scale analysis of cell signaling events. A number of studies have focused on tyrosine phosphorylation, an approach that has benefited from the ability to enrich for tyrosine phosphorylated peptides using peptide immunoprecipitation with anti-phosphotyrosine antibodies <xref ref-type="bibr" rid="pone.0021729-Salomon1">[24]</xref>&ndash;<xref ref-type="bibr" rid="pone.0021729-Cao1">[26]</xref>. While equivalent antibodies for phosphoserine and phosphothreonine have been employed previously <xref ref-type="bibr" rid="pone.0021729-Gronborg1">[27]</xref>, these antibodies did not prove useful for profiling protein phosphorylation in a highly complex sample.</p>
<p>Our goal of profiling mTORC1 signaling required that we refine and adapt available methods, and establish the reproducibility of our analyses. We combined several standard methods for protein and peptide fractionation to reduce sample complexity, thereby improving the sensitivity of the MS analysis <xref ref-type="bibr" rid="pone.0021729-Han1">[22]</xref>, <xref ref-type="bibr" rid="pone.0021729-Villen2">[28]</xref>. We also employed a newer strategy, the enrichment of phosphopeptides by using metal oxide chromatography (MOC). In this approach, phosphorylated peptides are enriched based on their affinity for metal oxides such as zirconium dioxide <xref ref-type="bibr" rid="pone.0021729-Kweon1">[29]</xref>, titanium dioxide <xref ref-type="bibr" rid="pone.0021729-Larsen1">[30]</xref> or aluminum hydroxide <xref ref-type="bibr" rid="pone.0021729-Wolschin1">[31]</xref>. Our studies used the more recently developed methodology of titanium dioxide (TiO<sub>2</sub>) chromatography in the presence of lactic acid and 2,5-dihydroxy benzoic acid (DHB) <xref ref-type="bibr" rid="pone.0021729-Hubbard1">[32]</xref>&ndash;<xref ref-type="bibr" rid="pone.0021729-Thingholm1">[34]</xref>.</p>
<p>Phosphoproteomic studies that have focused on signal transduction have largely been conducted using cell lines, and quantification of the greatest number of phosphorylation changes has been given primary importance over reproducibility of analysis. As an alternative to quantitative methods that employ isotope labeling, some investigators have employed &ldquo;label-free&rdquo; quantitation <xref ref-type="bibr" rid="pone.0021729-Huber1">[35]</xref>. However, data on the validity of this method for tissue analysis are very limited. We therefore took the approach of analyzing multiple technical and biological replicate samples using a phosphoproteomic platform that employs automated desalting and reversed-phase separation of peptides in a highly consistent and reproducible manner <xref ref-type="bibr" rid="pone.0021729-Yu1">[36]</xref>&ndash;<xref ref-type="bibr" rid="pone.0021729-Yu3">[38]</xref>. All column elutions and loading steps are accurately replicated through computer control <xref ref-type="bibr" rid="pone.0021729-Cao2">[39]</xref>.</p>
<p>Using the experimental approach described above, we were able to identify more than three thousand unique phosphorylation sites with high reproducibility on an LTQ-Fourier-transform ion cyclotron resonance (FTICR) classic mass spectrometer. By investigating the aforementioned <italic>in vivo</italic> model of mTORC1 activation and inhibition, we were able to identify a number of well characterized mTORC1 targets as well as previously unknown candidate phosphorylation events. Global analysis of our data provided for a broad characterization of hepatic mTORC1 signaling <italic>in vivo</italic>.</p>
</sec>
<sec id="s2" sec-type="methods">
<title>Methods</title>
<sec id="s2a">
<title>Materials</title>
<p>Rapamycin was purchased from LC Laboratories (Woburn, MA). Antibodies directed towards ribosomal protein S6 and phosphorylated S6 (Ser<sup>235/236</sup>) were from Cell Signaling Technology, Inc. (Danvers, MA). Sephadex G-25 Coarse, Q Sepharose Fast Flow, SP Sepharose Fast Flow, and Resource 15S media were purchased from GE Healthcare Bio-Sciences Corp. (Piscataway, NJ). Sequencing grade trypsin was from Promega Corp. (Madison, WI). Oasis HLB extraction cartridges were from Waters Corp. (Milford, MA). The Resource S column was purchased from GE Healthcare, Inc. (Piscataway, NJ). Titansphere Phos-TiO kits were obtained from GL Sciences, Inc. (Torrance, CA).</p>
</sec>
<sec id="s2b">
<title>Animal Studies, and Preparation and Processing of Liver Homogenates</title>
<p>Adult male Sprague-Dawley rats (Charles River Laboratories, Wilmington, MA) were used for all studies. In order to maximally activate signaling through mTORC1 <xref ref-type="bibr" rid="pone.0021729-Anand1">[17]</xref>, rats were fasted for 48 hr, at the end of which time they were administered vehicle (dimethyl sulfoxide; DMSO) or rapamycin (50 &micro;g/g body weight) by intraperitoneal injection. Fifteen minutes after injection, a 1 hr refeeding period was initiated by replacing the food in their cages. All rats were sacrificed by exsanguination under pentobarbital sodium anesthesia (50 &micro;g/g by intraperitoneal injection). Livers were flash-frozen in liquid nitrogen and stored at &minus;70&deg;C until use. Three rats per condition were studied. All animal studies were carried out in strict accordance with the recommendations in the Guide for the Care and Use of Laboratory Animals of the National Institutes of Health. The protocol was approved by the Institutional Animal Care and Use Committee of Rhode Island Hospital (CMTT# 0066-10).</p>
<p>Rat liver homogenates were prepared in 10 ml/g wet weight of Buffer A (50 mM β-glycerophosphate, pH 7.4, 150 mM NaCl, 100 &micro;M sodium orthovanadate, 10 mM NaF, 1 mM EGTA, and 5 mM EDTA) using a glass/Teflon homogenizer (7 strokes at 700 rpm). Homogenates were centrifuged at 1,000&times; <italic>g</italic> for 15 min. Supernatants were removed and centrifuged at 100,000&times; <italic>g</italic> for 1 hr. The resulting pellets were discarded. The supernatants retained and stored at &minus;70&deg;C.</p>
<p>Subsequent sample preparation was carried out at 4&deg;C. Upon thawing, homogenate (9 ml) was mixed with 18 ml of Buffer B (150 mM NaCl, 100 &micro;M sodium orthovanadate, 10 mM NaF, 1 mM EGTA, 5 mM EDTA). The diluted homogenate (25 ml) was applied to a 60 ml Sephadex G-25 Coarse column equilibrated in Buffer C (10 mM Tris, pH 8.6, 100 &micro;M sodium orthovanadate, 10 mM NaF, 0.5 mM EGTA, 2 mM EDTA). The column was eluted with 60 ml Buffer C. Fractions containing protein were collected and stored at &minus;70&deg;C until further analysis.</p>
<p>In order to fractionate homogenates into multiple samples for further analysis, strong anion exchange (SAX) separation was carried out on the gel filtered sample (50 mg protein) using a 9.4 mm&times;25 mm column packed with Q Sepharose Fast Flow beads. Following adsorption of the sample, the column was washed at 0.8 ml/min with 5 ml of Buffer C. The flow-through was collected. The column was then eluted stepwise at 0.8 ml/min with 5 ml of Buffer C containing 150 mM, 300 mM or 1 M sodium chloride. The SAX flow-through was adjusted to pH 7.3 then applied to a strong cation exchange (SCX) SP Sepharose column (9.4 mm&times;25 mm). The column was washed at 0.8 ml/min with Buffer D (Buffer C adjusted to pH 7.3) and the flow-through (30 ml) was collected. The column was then eluted with 10 ml of Buffer D containing 1 M NaCl. This combination of SAX and SCX chromatography yielded a total of 5 fractions.</p>
</sec>
<sec id="s2c">
<title>Peptide Preparation and Fractionation, and Purification of Phosphopeptides</title>
<p>Protein concentrations in the five fractions were measured using Bradford reagent. Samples containing 50 mg protein were denatured by adding urea to a final concentration of 8 M and dithiothreitol to a final concentration of 1 mM, followed by incubation for 1 hr at 56&deg;C. Samples were then alkylated with iodoacetemide (final concentration, 5 mM) followed by incubation for 1 hr at room temperature in the dark. These samples were diluted four-fold with 10 mM Tris, pH 8.0, and incubated overnight at 37&deg;C with sequencing grade modified trypsin at a 1∶100 protease∶protein ratio. The resulting tryptic peptides were adjusted to pH 2.7 with trifluoroacetic acid, and then cleared by centrifugation at 1000&times; <italic>g</italic> for 5 min. The peptide mixtures were desalted using Oasis HLB extraction cartridges and dried in a SpeedVac Concentrator.</p>
<p>Dry peptides were reconstituted in Buffer E (5 mM ammonium formate, pH 2.65, 30% acetonitrile) and loaded onto a 1 ml Resource S (SCX) column equilibrated with Buffer E. The column was eluted with a linear gradient of 5 mM to 1.5 M ammonium formate (flow rate of 0.5 ml/min) using an ISCO ProTeam LC chromatography system. Including the flow-through, 32 fractions (1 ml each) were collected. The peptide content of each fraction was estimated by measuring the absorbance at 280 nm. As described below, two fraction pools containing the highest peptide amounts were recovered from each chromatogram. These pools were dried in a SpeedVac Concentrator and stored at &minus;70&deg;C.</p>
<p>For further processing, each peptide SCX pool was dissolved in 400 &micro;l of 0.1% formic acid, 30% acetonitrile. These samples were each separated into four 100 &micro;l samples for the performance of technical replicates. Phosphopeptides were enriched using Titansphere Phos-TiO reagents. Adsorption proceeded according to the manufacturer's protocol. Phosphopeptides were eluted from the Phos-TiO tips by first applying 25 &micro;l of 1% ammonium hydroxide in water. The tips were then treated with 25 &micro;l of 1% ammonium hydroxide in 40% acetonitrile and finally eluted with 50 &micro;l of 5% pyrrolidine in water. All samples were dried in a SpeedVac Concentrator and stored at &minus;70&deg;C.</p>
</sec>
<sec id="s2d">
<title>Automated MS Analysis</title>
<p>Samples were analyzed by a fully automated phosphoproteomic technology platform that incorporates peptide desalting and separation via reverse phase chromatography followed by tandem mass spectrometry with static peak parking <xref ref-type="bibr" rid="pone.0021729-Nguyen1">[40]</xref>. Briefly, dry phosphopeptide-enriched mixtures were dissolved in 0.1% acetic acid in water and adsorbed to a C18 analytical column (360 &micro;m outer diameter&times;75 &micro;m inner diameter; fused silica with 12 cm of 5 &micro;m Monitor C18 particles and an integrated 4 &micro;m ESI emitter tip fritted with 3-&micro;m silica; Bangs Laboratories, Fishers, IN). Samples were eluted with a gradient of 0&ndash;70% 0.1 M acetic acid in acetonitrile that was developed over 30 min at a flow rate of 1.8 &micro;l/min. Resulting eluates were applied directly to the mass spectrometer (Linear Trap Quadrupole-Fourier Transform [LTQ-FT]; Thermo Fisher Scientific, Waltham, MA). As described previously, static peak parking was performed via flow rate reduction from 200 nl/min to 20 nl/min once peptides began to elute as determined from a bovine serum albumin peptide scouting run <xref ref-type="bibr" rid="pone.0021729-Ficarro1">[41]</xref>. The electrospray voltage of 2.0 kV was applied in a split flow configuration and spectra were collected in positive ion mode <xref ref-type="bibr" rid="pone.0021729-Cao2">[39]</xref>. One full MS scan in the Fourier Transform (from m/z 400 to 1800) was followed by 9 data-dependent MS/MS spectra in the linear ion trap from the 9 most abundant ions. Selected ions were dynamically excluded for 30 s and screened for charge-states of +1, +2 and +3. All other settings for FTMS and ion trap mass spectrometry scans were the same as described previously <xref ref-type="bibr" rid="pone.0021729-Nguyen1">[40]</xref>.</p>
</sec>
<sec id="s2e">
<title>Data Analysis and Peptide Quantification</title>
<p>Custom software was used for the analysis of all data. Automation of database searches, statistical validation of peptide sequence, analysis for phosphorylation site position, and uploading to a relational database was handled by our High Throughput Autonomous Proteomic Pipeline (HTAPP) <xref ref-type="bibr" rid="pone.0021729-Yu3">[38]</xref>. Quantitative proteomic data, including replicate analyses, was aggregated into heatmap representations using our PeptideDepot software <xref ref-type="bibr" rid="pone.0021729-Yu1">[36]</xref>.</p>
<p>Using our HTAPP software, MS/MS spectra were automatically searched against separate mouse, rat and hamster National Center for Biotechnology Information (NCBI) non-redundant protein databases (constructed on 7.30.2009) using the SEQUEST algorithm provided with Bioworks 3.2 (SEQUEST v.27 rev12). Peak lists were generated using extract_msn.exe version 4.0 using a mass range of 600&ndash;4500, precursor ion tolerance (for grouping) of 0.005 atomic mass unit, minimum ion count of 5, group scan of 0, minimum group count of 1. The NCBI rat, mouse and hamster databases contained 401,138 protein entries (50% forward, 50% reversed). SEQUEST was performed with the following parameters: trypsin enzyme specificity, 2 possible missed cleavages, 0.2 Da mass tolerance for precursor ions, and 0.5 Da mass tolerance for fragment ions. Search parameters specified a differential modification of phosphorylation (+79.9663 Da) on serine, threonine, and tyrosine residues and a static modification of carbamidomethylation (+57.0215 Da) on cysteine. To provide high confidence phosphopeptide sequence assignments, SEQUEST results were filtered by Xcorr (+1&gt;1.5; +2&gt;2.0; +3&gt;2.5), precursor mass error (&lt;20 ppm), and a logistic spectral score <xref ref-type="bibr" rid="pone.0021729-Yu2">[37]</xref> that assessed MS/MS spectral quality (&gt;0.965), minimum peak area threshold of 500. Non-redundant phosphopeptides and proteins with descriptors of &ldquo;unnamed&rdquo; or &ldquo;unknown&rdquo; were removed. False discovery rate was estimated with the decoy database approach after final assembly of nonredundant data into a comparison file <xref ref-type="bibr" rid="pone.0021729-Elias1">[42]</xref>. To validate the position of the phosphorylation site, the Ascore algorithm <xref ref-type="bibr" rid="pone.0021729-Beausoleil1">[43]</xref> was applied to all data. The reported phosphorylation site position reflected the top Ascore prediction. Ascore probabilities are reported in the full data table (<xref ref-type="supplementary-material" rid="pone.0021729.s001">Table S1</xref>).</p>
<p>Quantitative analysis, collation of replicates, and visualization of proteomic data was accomplished using PeptideDepot software <xref ref-type="bibr" rid="pone.0021729-Yu1">[36]</xref>. In order to compare peptide abundance among replicates, peak areas were calculated by inspection of selected ion chromatograms (SICs) using a software programmed in Microsoft Visual Basic 6.0 based on the Xcalibur Development Kit 2.0 SR2 (Thermo Fisher Scientific) <xref ref-type="bibr" rid="pone.0021729-Yu1">[36]</xref>. This approach used the ICIS algorithm available in the Xcalibur XDK with the following parameters: multiple resolutions of 8, noise tolerance of 0.1, noise window of 40, scans in baseline of 5, and inclusion of refexc peaks parameter value, which is false. Retention time alignment was performed to correct chromatographic shifts between runs. In brief, commonly observed peptides in each pair of liquid chromatography/mass spectrometry (LC/MS) runs were divided into 25 groups, evenly sorted by their retention time at 1% false discovery rate. The most abundant peptide within each group was selected as a landmark to align two MS runs. Quantitative data were calculated automatically for every assigned peptide in all LC/MS runs based on exact precursor mass and retention time. For the case in which a peptide was not confidently identified by MS/MS spectrum in a given LC/MS experiment, its retention time was predicted using the retention time observed in other LC/MS runs where the peptide was identified confidently through an MS/MS spectrum after alignment correction.</p>
<p>A label-free comparison data file was generated for the evaluation of phosphopeptide abundance in DMSO and rapamycin treated animals (<xref ref-type="supplementary-material" rid="pone.0021729.s001">Table S1</xref>). Based on an approach in which samples from control and rapamycin-treated animals were processed and analyzed in parallel, all comparisons were paired. That is, label-free ratios corresponding to peptide abundance differences between paired control and rapamycin animals for 3 biological replicate sets were calculated. Blanks in the data columns indicated that a clearly defined SIC peak was not observed for that phosphopeptide in any of the technical replicate analyses for that particular sample. The coefficient of variation was calculated for each animal sample among the 4 technical replicate analyses and 3 biological replicate analyses (<xref ref-type="supplementary-material" rid="pone.0021729.s001">Table S1</xref>).</p>
<p>A threshold for rapamycin effect was set at 5-fold. This degree of change was based on an analysis of the control∶rapamycin ratios for all phosphopeptides (<xref ref-type="supplementary-material" rid="pone.0021729.s002">Table S2</xref>).</p>
<p>To classify proteins from the liver phosphoproteome and rapamycin-sensitive candidates, gene ontology and pathway terms were examined (<ext-link ext-link-type="uri" xlink:href="http://www.geneontology.org">www.geneontology.org</ext-link>) using the PeptideDepot software <xref ref-type="bibr" rid="pone.0021729-Yu1">[36]</xref>. Classifications were based on the gene ontology slim terms and mapped to the Kyoto Encyclopedia of Genes and Genomes (KEGG) Pathway Database (<ext-link ext-link-type="uri" xlink:href="http://www.genome.ad.jp/kegg/pathway.html">http://www.genome.ad.jp/kegg/pathway.html</ext-link>). Phosphorylation site-specific kinase predictions were obtained using Kinexus' PhosphoNET kinase predictor software (<ext-link ext-link-type="uri" xlink:href="http://www.phosphonet.ca">http://www.phosphonet.ca</ext-link>).</p>
</sec>
</sec>
<sec id="s3">
<title>Results</title>
<sec id="s3a">
<title>Method Development</title>
<p>The profiling of mTORC1 signaling in liver required that we develop and validate methods for homogenate preparation, protein fractionation and peptide fractionation (<xref ref-type="fig" rid="pone-0021729-g001">Fig. 1</xref>). The first step in our strategy focused on removing phospholipids from rat liver homogenates. This was achieved by passing liver homogenates through a Sephadex G-25 gel filtration column. Gel electrophoresis and protein quantification showed greater than 90% protein recovery after this step.</p>
<fig id="pone-0021729-g001" position="float">
<object-id pub-id-type="doi">10.1371/journal.pone.0021729.g001</object-id>
<label>Figure 1</label>
<caption>
<title>Experimental approach for the phosphoproteomic profiling of liver homogenates.</title>
<p>Six rats were fasted for 48 hr, and then injected with DMSO or rapamycin. A 1 hr refeeding period was initiated 15 min after injection. As shown in this flow diagram, tissue homogenates were processed by sequential gel filtration, SAX and SCX chromatography. Samples were then reduced, alkylated, and digested into peptides. Tryptic peptides were desalted using Oasis HLB cartridges and further separated into two fraction pools by SCX chromatography. Phosphopeptides were then enriched by TiO<sub>2</sub> chromatography. Purified phosphopeptides were analyzed by reversed-phase LC-MS/MS.</p>
</caption>
<graphic xlink:href="pone.0021729.g001"></graphic>
</fig>
<p>In order to fractionate the sample to enhance MS phosphopeptide detection, ion exchange chromatography was performed following gel filtration. First, SAX chromatography with step elution generated a flow-through fraction and fractions containing proteins eluted with 0.15 M NaCl, 0.3 M NaCl and 1 M NaCl. The SAX flow-through was applied to a SCX column, resulting in 2 additional fractions, a SCX flow-through fraction and the proteins eluted at 1 M NaCl. SDS-PAGE of the resulting ion exchange fractions (not shown) demonstrated distinct patterns of protein expression. As intended, protein content of the five fractions was similar.</p>
<p>The five ion exchange fractions were digested with trypsin. The resultant peptides were desalted by solid-phase extraction and then further separated by peptide SCX chromatography (<xref ref-type="fig" rid="pone-0021729-g002">Fig. 2</xref>, <italic>Upper Panel</italic>). Two fraction pools containing the highest peptide amounts were collected from each column run. Each peptide SCX pool was separated into 4 samples for the performance of technical replicates. Phosphopeptides were enriched from each of these replicate pools by TiO<sub>2</sub> affinity purification.</p>
<fig id="pone-0021729-g002" position="float">
<object-id pub-id-type="doi">10.1371/journal.pone.0021729.g002</object-id>
<label>Figure 2</label>
<caption>
<title>Phosphopeptide enrichment by SCX Chromatography.</title>
<p>
<italic>Upper Panel</italic>: Peptide SCX chromatography separation at pH 2.65 of a liver homogenate fraction. Digested peptides (10 mg) were applied to a Resource S SCX column and eluted using a 5 mM to 1.5 M ammonium formate gradient. The peptide content of each fraction was estimated by measuring the absorbance at 280 nm. Two fraction pools containing the highest peptide amounts (Peaks 1 and 2) were recovered from each column run. The figure shows a representative result using the 300 mM NaCl SAX fraction of the control 1 liver sample. <italic>Lower Panel</italic>: The number of phosphopeptides identified from peak 1 (filled bars) versus peak 2 (unfilled bars) is shown as a function of whether those phosphopeptides contained one, two or three phosphorylation sites.</p>
</caption>
<graphic xlink:href="pone.0021729.g002"></graphic>
</fig>
<p>The efficacy of the peptide SCX chromatography was examined by comparing the phosphopeptides present in peak 1 versus peak 2. Since each phosphate group subtracts a net charge of 1 from a peptide at pH 2.65, we expected phosphorylated peptides to elute before non-phosphorylated peptides <xref ref-type="bibr" rid="pone.0021729-Beausoleil2">[44]</xref>. MS analysis confirmed that fractions from SCX peak 1 were highly enriched with phosphopeptides and successfully separated from the more complex, positively-charged, multivalent peptides that eluted in the second SCX peak (<xref ref-type="fig" rid="pone-0021729-g002">Fig. 2</xref>, <italic>Lower Panel</italic>). Following TiO<sub>2</sub> chromatography, but prior to filtering of MS results for non-phosphorylated peptides, more than 90% of the peptides recovered from SCX peak 1 were phosphorylated. In comparison, the yield of phosphopeptides from SCX peak 2 was approximately 60%. Most importantly, this chromatography step further reduced the complexity of the samples for MS analysis.</p>
<p>The use of TiO<sub>2</sub> affinity phosphopeptide enrichment was based on previous studies demonstrating its high phosphopeptide selectivity <xref ref-type="bibr" rid="pone.0021729-Larsen1">[30]</xref>, <xref ref-type="bibr" rid="pone.0021729-Olsen1">[33]</xref>. TiO<sub>2</sub> enrichment using loose TiO<sub>2</sub> beads in the presence of 2,5-dihydroxybenzoic acid <xref ref-type="bibr" rid="pone.0021729-Larsen1">[30]</xref> provided for a modest improvement in phosphopeptide enrichment relative to initial analyses using Fe<sup>3+</sup> immobilized metal affinity chromatography (IMAC). The lactic acid-treated Titansphere Phos-TiO kit was tested based on a report that it yields superior results <xref ref-type="bibr" rid="pone.0021729-Sugiyama1">[45]</xref>. A pyrrolidine solution along with 1% ammonium hydroxide solution <xref ref-type="bibr" rid="pone.0021729-Kyono1">[46]</xref> was also employed to further improve the efficiency of phosphopeptide elution from TiO<sub>2</sub> beads.</p>
<p>Phosphopeptide enriched samples were analyzed by LC/MS as described above. The resulting MS/MS spectra (can be accessed at <ext-link ext-link-type="uri" xlink:href="http://cellpathway.com/SupplementalMaterial3.zip">http://cellpathway.com/SupplementalMaterial3.zip</ext-link> and <ext-link ext-link-type="uri" xlink:href="http://cellpathway.com/SupplementalMaterial4.zip">http://cellpathway.com/SupplementalMaterial4.zip</ext-link>) were searched against the mouse, hamster and rat subsets of NCBI non-redundant databases using the SEQUEST algorithm. Sequence assignments were made using stringent filtering criteria (logistic spectral score &gt;0.965 <xref ref-type="bibr" rid="pone.0021729-Yu1">[36]</xref>, <xref ref-type="bibr" rid="pone.0021729-Nguyen1">[40]</xref>; precursor mass error &lt;20 ppm; minimum SIC peak area threshold of 500 for label-free quantitation). After filtering, the false discovery rate, estimated using a decoy database <xref ref-type="bibr" rid="pone.0021729-Moser1">[20]</xref>, <xref ref-type="bibr" rid="pone.0021729-Yu2">[37]</xref>, was 1%.</p>
<p>The final data set derived from three biological control replicates contained 3,231 nonredundant phosphorylation sites representing 1,400 proteins and derived from 4,238 phosphopeptides. To assess the reproducibility of our phosphopeptide enrichment strategy in the context of <italic>in vivo</italic> studies, we compared LC/MS runs from these three control biological replicates as well as four technical replicates within each biological replicate. LC/MS retention time alignments of multiple datasets were carried out to correct for chromatographic shifts between experiments using the PeptideDepot software. Eighty-two percent of the phosphopeptides were identified in all three control biological replicates. Only 13% and 5% of the phosphopeptides were identified in two and one of the control biological replicates, respectively. Analysis of technical replicates demonstrated that, on average, 40% of the phosphopeptides were identified in all four technical replicates in a given biological sample, while 27%, 20% and 13% of the phosphopeptides were identified in three, two and one of the technical replicates, respectively. Thus, the performance of the four technical replicates contributed greatly to the reproducibility of the biological replicates.</p>
<p>To evaluate the sensitivity of our method for relatively low abundance phosphopeptides, we examined the relationship between peak area and the probability of a phosphopeptide being detected in one, two or all three of the biological replicates. Results (not shown) confirmed that more abundant phosphopeptides were indeed identified with greater reproducibility.</p>
</sec>
<sec id="s3b">
<title>The Rat Liver Phosphoproteome</title>
<p>To characterize the rat liver phosphoproteome from animals in which hepatic signal transduction was nutrient-activated, we compiled a list of all identified phosphorylation sites generated by analysis of the six combined control and rapamycin-treated animals. A total of 3,234 unique phosphorylation sites representing 1,401 proteins and 4,241 phosphopeptides were identified from these samples (<xref ref-type="supplementary-material" rid="pone.0021729.s001">Table S1</xref>). Serine phosphorylation predominated (2,762 sites representing 85% of all phosphorylation sites). Serine sites were followed in abundance by threonine phosphorylation (415 sites; 13%) and tyrosine phosphorylation (57 sites; 2%). Close to certainty (&gt;99%) of phosphorylation site localization was achieved for 71% of the data set (3,011 of 4,241 phosphopeptides). All phosphopeptides with ambiguous site assignments (Ascore &lt;19) <xref ref-type="bibr" rid="pone.0021729-Beausoleil1">[43]</xref> are presented in a separate table (<xref ref-type="supplementary-material" rid="pone.0021729.s001">Table S1</xref>).</p>
</sec>
<sec id="s3c">
<title>Rapamycin-regulated Phosphorylation Events</title>
<p>Prior to performing the phosphoproteomic analysis, the efficacy of rapamycin administration was assessed by phospho-specific Western immunoblotting for ribosomal protein S6 (Ser235 and Ser236) <xref ref-type="bibr" rid="pone.0021729-Anand2">[18]</xref>. Results (<xref ref-type="fig" rid="pone-0021729-g003">Fig. 3</xref>) showed a greater than 10-fold reduction in animals that were administered rapamycin prior to refeeding.</p>
<fig id="pone-0021729-g003" position="float">
<object-id pub-id-type="doi">10.1371/journal.pone.0021729.g003</object-id>
<label>Figure 3</label>
<caption>
<title>Demonstration of rapamycin <italic>in vivo</italic> effect.</title>
<p>Extracts were prepared using liver samples from three control (C) and three rapamycin-treated (R) rats. The extracts (80 &micro;g protein per lane) were analyzed by direct immunoblotting for phosphorylated ribosomal protein S6 (P-S6) and total S6.</p>
</caption>
<graphic xlink:href="pone.0021729.g003"></graphic>
</fig>
<p>To identify rapamycin-modulated phosphorylation events involving soluble proteins from rat liver, we compared the results derived from animals that received DMSO vehicle versus rapamycin prior to refeeding. The magnitude of the phosphorylation change for each phosphopeptide in each paired analysis was determined using PeptideDepot software to generate mean peak area ratios. An analysis of the number of peptides showing graded degrees of change <xref ref-type="bibr" rid="pone.0021729-Bronshtein1">[47]</xref> showed an inflection point between 4-fold and 5-fold (<xref ref-type="supplementary-material" rid="pone.0021729.s002">Table S2</xref> and <xref ref-type="fig" rid="pone-0021729-g004">Fig. 4</xref>). Similar to the approach taken by Chen et al. <xref ref-type="bibr" rid="pone.0021729-Chen1">[48]</xref>, we used this information to assign a conservative threshold of a 5-fold change as indicating an effect of rapamycin. That is, we applied a cutoff of &gt;5 or &lt;0.2 (ratio of control∶rapamycin) for sites whose abundance decreased or increased, respectively. Using these criteria, we identified 67 unique sites that were altered in response to rapamycin out of the total of 3,234 sites identified.</p>
<fig id="pone-0021729-g004" position="float">
<object-id pub-id-type="doi">10.1371/journal.pone.0021729.g004</object-id>
<label>Figure 4</label>
<caption>
<title>Distribution of rapamycin-induced fold-change in phosphopeptide abundance.</title>
<p>For each peptide identified in all three control samples, fold-change in response to rapamycin was determined to be the minimum fold-change among the three paired analyses generated for each phosphopeptide. The graph shows the number of peptides in categories ranging from less than 1.1-fold different to 40-fold different or greater.</p>
</caption>
<graphic xlink:href="pone.0021729.g004"></graphic>
</fig>
<p>Of these 67 sites, 5 have been previously reported as downstream from mTORC1. These include Maf1 (Ser75) and raptor (Ser863), both of which were identified only recently <xref ref-type="bibr" rid="pone.0021729-Foster2">[49]</xref>, <xref ref-type="bibr" rid="pone.0021729-Shor1">[50]</xref>. The other three rapamycin-sensitive sites were identified on the established mTORC1 downstream target, ribosomal protein S6. Consistent with immunoblotting results, S6 phosphorylation at Ser235 and Ser236 was reduced at least 10-fold in response to rapamycin in all three paired samples.</p>
<p>An unexpected observation was that the 62 phosphorylation sites not previously identified as rapamycin-sensitive were both up-regulated and down-regulated in association with rapamycin administration. Twenty-nine phosphorylation sites were reduced in abundance at least 5-fold following rapamycin administration (<xref ref-type="table" rid="pone-0021729-t001">Table 1</xref>), while the other 33 rapamycin-sensitive phosphorylation sites were increased at least 5-fold in samples in response to rapamycin (<xref ref-type="table" rid="pone-0021729-t002">Table 2</xref>). No tyrosine phosphorylated sites were identified as rapamycin-regulated.</p>
<table-wrap id="pone-0021729-t001" position="float">
<object-id pub-id-type="doi">10.1371/journal.pone.0021729.t001</object-id>
<label>Table 1</label>
<caption>
<title>Phosphorylation sites down-regulated in response to rapamycin.</title>
</caption>
<alternatives>
<graphic id="pone-0021729-t001-1" xlink:href="pone.0021729.t001"></graphic>
<table frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
</colgroup>
<thead>
<tr>
<td align="left" colspan="1" rowspan="1">Protein name</td><td align="left" colspan="1" rowspan="1">Gene Symbol</td><td align="left" colspan="1" rowspan="1">Phosphosite</td><td align="left" colspan="1" rowspan="1">C/R Ratio 1</td><td align="left" colspan="1" rowspan="1">C/R Ratio 2</td><td align="left" colspan="1" rowspan="1">C/R Ratio 3</td>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">AKT1 substrate 1 (PRAS40) isoform CRA_d</td><td align="left" colspan="1" rowspan="1">AKT1S1</td><td align="left" colspan="1" rowspan="1">S203/S213</td><td align="left" colspan="1" rowspan="1">79</td><td align="left" colspan="1" rowspan="1">9</td><td align="left" colspan="1" rowspan="1">&gt;100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AMP-activated protein kinase alpha-2</td><td align="left" colspan="1" rowspan="1">PRKAA2</td><td align="left" colspan="1" rowspan="1">S377</td><td align="left" colspan="1" rowspan="1">22</td><td align="left" colspan="1" rowspan="1">96</td><td align="left" colspan="1" rowspan="1">109</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Autophagy-related protein 2 homolog A (rCG47388)</td><td align="left" colspan="1" rowspan="1">ATG2A</td><td align="left" colspan="1" rowspan="1">S1243</td><td align="left" colspan="1" rowspan="1">16</td><td align="left" colspan="1" rowspan="1">38</td><td align="left" colspan="1" rowspan="1">218</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Bcl2-associated athanogene 3</td><td align="left" colspan="1" rowspan="1">BAG3</td><td align="left" colspan="1" rowspan="1">S174</td><td align="left" colspan="1" rowspan="1">12</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Carbomyl-phosphate synthase 2</td><td align="left" colspan="1" rowspan="1">CAD</td><td align="left" colspan="1" rowspan="1">S1859</td><td align="left" colspan="1" rowspan="1">9</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">&gt;100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Delta-4-3-ketosteroid 5-beta-reductase</td><td align="left" colspan="1" rowspan="1">AKR1D1</td><td align="left" colspan="1" rowspan="1">S235</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">&gt;100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eukaryotic translation initiation factor 3A (ZH12 protein)</td><td align="left" colspan="1" rowspan="1">EIF3A</td><td align="left" colspan="1" rowspan="1">S584</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">&gt;100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Fam65a protein</td><td align="left" colspan="1" rowspan="1">FAM65A</td><td align="left" colspan="1" rowspan="1">S450</td><td align="left" colspan="1" rowspan="1">12</td><td align="left" colspan="1" rowspan="1">59</td><td align="left" colspan="1" rowspan="1">5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Family with sequence similarity 126, member B</td><td align="left" colspan="1" rowspan="1">FAM126B</td><td align="left" colspan="1" rowspan="1">T306</td><td align="left" colspan="1" rowspan="1">6</td><td align="left" colspan="1" rowspan="1">17</td><td align="left" colspan="1" rowspan="1">13</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Growth factor receptor binding protein 7</td><td align="left" colspan="1" rowspan="1">GRB7</td><td align="left" colspan="1" rowspan="1">S364</td><td align="left" colspan="1" rowspan="1">21</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">&gt;100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Hepatoma-derived growth factor 3 (HDGF2)</td><td align="left" colspan="1" rowspan="1">HDGFRP2</td><td align="left" colspan="1" rowspan="1">S450</td><td align="left" colspan="1" rowspan="1">65</td><td align="left" colspan="1" rowspan="1">77</td><td align="left" colspan="1" rowspan="1">7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Liver regeneration-related protein LRRG07</td><td align="left" colspan="1" rowspan="1">AKR1C13</td><td align="left" colspan="1" rowspan="1">S129</td><td align="left" colspan="1" rowspan="1">7</td><td align="left" colspan="1" rowspan="1">14</td><td align="left" colspan="1" rowspan="1">8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MAF1 (Repressor of RNA polymerase III transcription)</td><td align="left" colspan="1" rowspan="1">MAF1</td><td align="left" colspan="1" rowspan="1">S75</td><td align="left" colspan="1" rowspan="1">11</td><td align="left" colspan="1" rowspan="1">29</td><td align="left" colspan="1" rowspan="1">37</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mCG147067 protein [Mus musculus]</td><td align="left" colspan="1" rowspan="1">N/A</td><td align="left" colspan="1" rowspan="1">S76</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">32</td><td align="left" colspan="1" rowspan="1">83</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Metastasis suppressor 1, isoform CRA_a</td><td align="left" colspan="1" rowspan="1">MTSS1</td><td align="left" colspan="1" rowspan="1">S667</td><td align="left" colspan="1" rowspan="1">12</td><td align="left" colspan="1" rowspan="1">12</td><td align="left" colspan="1" rowspan="1">93</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Peroxisome biogenesis factor 1</td><td align="left" colspan="1" rowspan="1">PEX1</td><td align="left" colspan="1" rowspan="1">S1181</td><td align="left" colspan="1" rowspan="1">7</td><td align="left" colspan="1" rowspan="1">8</td><td align="left" colspan="1" rowspan="1">30</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Protein kinase C beta II</td><td align="left" colspan="1" rowspan="1">PRKCB</td><td align="left" colspan="1" rowspan="1">S660</td><td align="left" colspan="1" rowspan="1">80</td><td align="left" colspan="1" rowspan="1">21</td><td align="left" colspan="1" rowspan="1">14</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Protocadherin alpha 4 homolog</td><td align="left" colspan="1" rowspan="1">PCDHA4</td><td align="left" colspan="1" rowspan="1">S145</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">53</td><td align="left" colspan="1" rowspan="1">&gt;100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PTPRF interacting protein (liprin) alpha 1</td><td align="left" colspan="1" rowspan="1">PPFIA1</td><td align="left" colspan="1" rowspan="1">S667</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">14</td><td align="left" colspan="1" rowspan="1">12</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Raptor</td><td align="left" colspan="1" rowspan="1">RPTR</td><td align="left" colspan="1" rowspan="1">S863</td><td align="left" colspan="1" rowspan="1">5</td><td align="left" colspan="1" rowspan="1">21</td><td align="left" colspan="1" rowspan="1">17</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rCG21490, isoform CRA_a (LOC687565 protein)</td><td align="left" colspan="1" rowspan="1">LOC687565</td><td align="left" colspan="1" rowspan="1">T74</td><td align="left" colspan="1" rowspan="1">17</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">&gt;100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rCG50767 (Similar to La-related protein 4)</td><td align="left" colspan="1" rowspan="1">LOC683071</td><td align="left" colspan="1" rowspan="1">S379</td><td align="left" colspan="1" rowspan="1">26</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">&gt;100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ribosomal protein S6</td><td align="left" colspan="1" rowspan="1">RPS6</td><td align="left" colspan="1" rowspan="1">S235/S236</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">&gt;100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ribosomal protein S6</td><td align="left" colspan="1" rowspan="1">RPS6</td><td align="left" colspan="1" rowspan="1">S235/S236/S240</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">21</td><td align="left" colspan="1" rowspan="1">&gt;100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ribosomal protein S6</td><td align="left" colspan="1" rowspan="1">RPS6</td><td align="left" colspan="1" rowspan="1">S236/S240</td><td align="left" colspan="1" rowspan="1">51</td><td align="left" colspan="1" rowspan="1">87</td><td align="left" colspan="1" rowspan="1">&gt;100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Similar to AI661453 protein (C6orf132)</td><td align="left" colspan="1" rowspan="1">RGD1561662</td><td align="left" colspan="1" rowspan="1">S680</td><td align="left" colspan="1" rowspan="1">5</td><td align="left" colspan="1" rowspan="1">9</td><td align="left" colspan="1" rowspan="1">49</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Similar to hypothetical protein 4933430I17</td><td align="left" colspan="1" rowspan="1">LOC500475</td><td align="left" colspan="1" rowspan="1">S5S6</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">&gt;100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Sugen Kinase 269 (hypothetical protein XP_236266)</td><td align="left" colspan="1" rowspan="1">RGD1312026</td><td align="left" colspan="1" rowspan="1">S281</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">&gt;100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ubiquitin specific peptidase 15, isoform CRA_a</td><td align="left" colspan="1" rowspan="1">USP15</td><td align="left" colspan="1" rowspan="1">S244</td><td align="left" colspan="1" rowspan="1">35</td><td align="left" colspan="1" rowspan="1">79</td><td align="left" colspan="1" rowspan="1">77</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ubiquitin specific peptidase 15, isoform CRA_a</td><td align="left" colspan="1" rowspan="1">USP15</td><td align="left" colspan="1" rowspan="1">S242</td><td align="left" colspan="1" rowspan="1">31</td><td align="left" colspan="1" rowspan="1">84</td><td align="left" colspan="1" rowspan="1">44</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ubiquitin specific protease 24</td><td align="left" colspan="1" rowspan="1">USP24</td><td align="left" colspan="1" rowspan="1">S1138</td><td align="left" colspan="1" rowspan="1">&gt;100</td><td align="left" colspan="1" rowspan="1">27</td><td align="left" colspan="1" rowspan="1">21</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WASH complex subunit FAM21 (NP61201)</td><td align="left" colspan="1" rowspan="1">FAM21C</td><td align="left" colspan="1" rowspan="1">S744</td><td align="left" colspan="1" rowspan="1">40</td><td align="left" colspan="1" rowspan="1">10</td><td align="left" colspan="1" rowspan="1">12</td>
</tr>
</tbody>
</table>
</alternatives>
</table-wrap>
<table-wrap id="pone-0021729-t002" position="float">
<object-id pub-id-type="doi">10.1371/journal.pone.0021729.t002</object-id>
<label>Table 2</label>
<caption>
<title>Phosphorylation sites up-regulated in response to rapamycin.</title>
</caption>
<alternatives>
<graphic id="pone-0021729-t002-2" xlink:href="pone.0021729.t002"></graphic>
<table frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
</colgroup>
<thead>
<tr>
<td align="left" colspan="1" rowspan="1">Protein name</td><td align="left" colspan="1" rowspan="1">Gene Symbol</td><td align="left" colspan="1" rowspan="1">Phosphosite</td><td align="left" colspan="1" rowspan="1">C/R Ratio 1</td><td align="left" colspan="1" rowspan="1">C/R Ratio 2</td><td align="left" colspan="1" rowspan="1">C/R Ratio 3</td>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Acyl-CoA thioesterase 1</td><td align="left" colspan="1" rowspan="1">ACOT1</td><td align="left" colspan="1" rowspan="1">S416</td><td align="left" colspan="1" rowspan="1">0.03</td><td align="left" colspan="1" rowspan="1">0.06</td><td align="left" colspan="1" rowspan="1">0.16</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ajuba protein</td><td align="left" colspan="1" rowspan="1">JUB</td><td align="left" colspan="1" rowspan="1">S147</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">0.04</td><td align="left" colspan="1" rowspan="1">0.09</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Arginase 1, liver</td><td align="left" colspan="1" rowspan="1">ARG1</td><td align="left" colspan="1" rowspan="1">T281</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">0.01</td><td align="left" colspan="1" rowspan="1">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Argininosuccinate synthetase</td><td align="left" colspan="1" rowspan="1">ASS1</td><td align="left" colspan="1" rowspan="1">T219</td><td align="left" colspan="1" rowspan="1">0.08</td><td align="left" colspan="1" rowspan="1">0.03</td><td align="left" colspan="1" rowspan="1">0.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Carbohydrate responsive element binding protein</td><td align="left" colspan="1" rowspan="1">MLXIPL</td><td align="left" colspan="1" rowspan="1">S516</td><td align="left" colspan="1" rowspan="1">0.17</td><td align="left" colspan="1" rowspan="1">0.12</td><td align="left" colspan="1" rowspan="1">0.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cytosolic 3-hydroxy 3-methylglutaryl coenzyme A synthase</td><td align="left" colspan="1" rowspan="1">HMGCS1</td><td align="left" colspan="1" rowspan="1">S516</td><td align="left" colspan="1" rowspan="1">0.05</td><td align="left" colspan="1" rowspan="1">0.1</td><td align="left" colspan="1" rowspan="1">0.15</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cytosolic malate dehydrogenase</td><td align="left" colspan="1" rowspan="1">MDH1</td><td align="left" colspan="1" rowspan="1">(S188/S189)</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">0.04</td><td align="left" colspan="1" rowspan="1">0.13</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ERM-binding phosphoprotein (NHERF)</td><td align="left" colspan="1" rowspan="1">SLC9A3R1</td><td align="left" colspan="1" rowspan="1">S287T290/S291</td><td align="left" colspan="1" rowspan="1">0.07</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Estrogen receptor-binding site associated antigen 9</td><td align="left" colspan="1" rowspan="1">EBAG9</td><td align="left" colspan="1" rowspan="1">S36</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">0.07</td><td align="left" colspan="1" rowspan="1">0.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Family with sequence similarity 83, member H</td><td align="left" colspan="1" rowspan="1">FAM83H</td><td align="left" colspan="1" rowspan="1">S871</td><td align="left" colspan="1" rowspan="1">0.01</td><td align="left" colspan="1" rowspan="1">0.15</td><td align="left" colspan="1" rowspan="1">0.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GABA-B receptor-interacting scaffolding protein</td><td align="left" colspan="1" rowspan="1">AKAP9</td><td align="left" colspan="1" rowspan="1">S104</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">0.08</td><td align="left" colspan="1" rowspan="1">0.11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAPDH - Glyceraldehyde-3-phosphate dehydrogenase</td><td align="left" colspan="1" rowspan="1">GAPDH</td><td align="left" colspan="1" rowspan="1">T182</td><td align="left" colspan="1" rowspan="1">0.11</td><td align="left" colspan="1" rowspan="1">0.14</td><td align="left" colspan="1" rowspan="1">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Gephyrin</td><td align="left" colspan="1" rowspan="1">GPHN</td><td align="left" colspan="1" rowspan="1">S200</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">0.08</td><td align="left" colspan="1" rowspan="1">0.05</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Inositol polyphosphate 5-phosphatase (Sec16 hom. A)</td><td align="left" colspan="1" rowspan="1">INNP5E</td><td align="left" colspan="1" rowspan="1">S711</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">0.14</td><td align="left" colspan="1" rowspan="1">0.19</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">JTV1 protein (Aimp2)</td><td align="left" colspan="1" rowspan="1">AIMP2</td><td align="left" colspan="1" rowspan="1">T35</td><td align="left" colspan="1" rowspan="1">0.03</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">l-Afadin (AF6)</td><td align="left" colspan="1" rowspan="1">MLLT4</td><td align="left" colspan="1" rowspan="1">S1804</td><td align="left" colspan="1" rowspan="1">0.03</td><td align="left" colspan="1" rowspan="1">0.06</td><td align="left" colspan="1" rowspan="1">0.14</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">La ribonucleoprotein domain family, member 1</td><td align="left" colspan="1" rowspan="1">LARP1</td><td align="left" colspan="1" rowspan="1">S648</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">0.19</td><td align="left" colspan="1" rowspan="1">0.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">La ribonucleoprotein domain family, member 1</td><td align="left" colspan="1" rowspan="1">LARP1</td><td align="left" colspan="1" rowspan="1">T644</td><td align="left" colspan="1" rowspan="1">0.01</td><td align="left" colspan="1" rowspan="1">0.19</td><td align="left" colspan="1" rowspan="1">0.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Liver regeneration-related protein LRRG07</td><td align="left" colspan="1" rowspan="1">AKR1C13</td><td align="left" colspan="1" rowspan="1">S232</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">0.13</td><td align="left" colspan="1" rowspan="1">0.08</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Nuclear factor 1/A</td><td align="left" colspan="1" rowspan="1">NFIA</td><td align="left" colspan="1" rowspan="1">S280</td><td align="left" colspan="1" rowspan="1">0.06</td><td align="left" colspan="1" rowspan="1">0.02</td><td align="left" colspan="1" rowspan="1">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Phosphoglucomutase 1</td><td align="left" colspan="1" rowspan="1">PGM1</td><td align="left" colspan="1" rowspan="1">T115</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">0.05</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Phosphoglucomutase 1</td><td align="left" colspan="1" rowspan="1">PGM1</td><td align="left" colspan="1" rowspan="1">S117</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Phosphoglycerate kinase 1</td><td align="left" colspan="1" rowspan="1">PGK1</td><td align="left" colspan="1" rowspan="1">S203</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">0.1</td><td align="left" colspan="1" rowspan="1">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Plakophilin 4</td><td align="left" colspan="1" rowspan="1">PKP4</td><td align="left" colspan="1" rowspan="1">S335</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">0.14</td><td align="left" colspan="1" rowspan="1">0.14</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rCG23015, isoform CRA_a [Rattus norvegicus] [107894 Da]</td><td align="left" colspan="1" rowspan="1">N/A</td><td align="left" colspan="1" rowspan="1">S356</td><td align="left" colspan="1" rowspan="1">0.01</td><td align="left" colspan="1" rowspan="1">0.06</td><td align="left" colspan="1" rowspan="1">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rCG35745 (LOC679383 protein)</td><td align="left" colspan="1" rowspan="1">LOC679383</td><td align="left" colspan="1" rowspan="1">S469</td><td align="left" colspan="1" rowspan="1">0.17</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ribonuclease UK114 (perchrolic acid soluble protein)</td><td align="left" colspan="1" rowspan="1">HRSP12</td><td align="left" colspan="1" rowspan="1">S11</td><td align="left" colspan="1" rowspan="1">0.03</td><td align="left" colspan="1" rowspan="1">0.1</td><td align="left" colspan="1" rowspan="1">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ribonuclease UK114 (perchrolic acid soluble protein)</td><td align="left" colspan="1" rowspan="1">HRSP12</td><td align="left" colspan="1" rowspan="1">T10</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ribosome binding protein 1</td><td align="left" colspan="1" rowspan="1">RRBP1</td><td align="left" colspan="1" rowspan="1">S1203</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td><td align="left" colspan="1" rowspan="1">0.18</td><td align="left" colspan="1" rowspan="1">&lt;0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">S6 protein kinase (p90-RSK1)</td><td align="left" colspan="1" rowspan="1">RPS6KA1</td><td align="left" colspan="1" rowspan="1">S363</td><td align="left" colspan="1" rowspan="1">0.04</td><td align="left" colspan="1" rowspan="1">0.15</td><td align="left" colspan="1" rowspan="1">0.05</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Stromal interaction molecule 1</td><td align="left" colspan="1" rowspan="1">STIM1</td><td align="left" colspan="1" rowspan="1">S519</td><td align="left" colspan="1" rowspan="1">0.01</td><td align="left" colspan="1" rowspan="1">0.2</td><td align="left" colspan="1" rowspan="1">0.02</td>
</tr>
</tbody>
</table>
</alternatives>
</table-wrap>
<p>Several phosphorylation events were particularly noteworthy. Phosphorylation of AMP kinase 2 (AMPK2, Ser377), eukaryotic initiation factor 3a (eIF3a; Ser584), metastasis suppressor 1 (Ser667), protein kinase C beta (Ser660), and PRAS40 (Ser203, Ser213) were down-regulated in response to rapamycin. The protein kinase C beta site has been shown to regulate the cellular localization of this kinase <xref ref-type="bibr" rid="pone.0021729-Keranen1">[51]</xref>. Ser 363 on p90 S6 kinase 1, which was up-regulated, is one of the known, multiple phosphorylation sites required for activation of this kinase <xref ref-type="bibr" rid="pone.0021729-Dalby1">[52]</xref>. In addition, PRAS40 (Ser213) was previously shown to be phosphorylated by mTORC1 in vitro. However, phosphorylation of this site was shown not to be sensitive to rapamycin treatment in vivo by using mutated PRAS40 protein in HEK293 cells <xref ref-type="bibr" rid="pone.0021729-Wang1">[53]</xref>. The remaining phosphorylation sites have not been characterized with regard to function, although gephyrin, PRAS40 and AMPK were shown previously to interact with mTORC1 <xref ref-type="bibr" rid="pone.0021729-Sabatini1">[54]</xref>&ndash;<xref ref-type="bibr" rid="pone.0021729-Vander1">[56]</xref>.</p>
</sec>
<sec id="s3d">
<title>Gene Ontology and Pathway Analysis</title>
<p>We profiled the proteins identified in the phosphoproteomic analysis from control rat livers and compared this dataset to the phosphoproteins affected by rapamycin administration. To further characterize rapamycin action in liver, we used the Gene Ontology metadata in the Human Protein Reference Database (HPRD) <xref ref-type="bibr" rid="pone.0021729-Prasad1">[57]</xref>. For this analysis of rapamycin effect, we broadened the array of phosphoproteins by including those whose phosphorylation state changed by &gt;5-fold in two of three biological replicates and at least 3-fold in the third replicate. This led to the categorization of 81 rapamycin-regulated candidate proteins.</p>
<p>Categorization of the rapamycin-responsive phosphoproteome for biological processes (<xref ref-type="fig" rid="pone-0021729-g005">Fig. 5</xref>, <italic>top panel</italic>) showed that 10.5% of the rapamycin-responsive candidate proteins were involved in the regulation of transcription, 7% in oxidation reduction, and 4.7% in translation. In the whole rat liver phosphoproteome, the same biological processes were similarly ascribed to 9.4%, 2.7%, and 2.0%, respectively. Categorization of phosphoproteins by cellular localization (<xref ref-type="fig" rid="pone-0021729-g005">Fig. 5</xref>. <italic>third panel</italic>) showed significant under-representation of the &ldquo;mitochondrion&rdquo; category among rapamycin-sensitive phosphoproteins. Categorization by molecular processes of all phosphoproteins detected (<xref ref-type="fig" rid="pone-0021729-g005">Fig. 5</xref>, <italic>second panel</italic>) showed dominance of &ldquo;protein binding.&rdquo; There were no significant differences when compared to the distribution of phosphoproteins that were affected by rapamycin.</p>
<fig id="pone-0021729-g005" position="float">
<object-id pub-id-type="doi">10.1371/journal.pone.0021729.g005</object-id>
<label>Figure 5</label>
<caption>
<title>Gene ontology distribution of phosphoproteins in the total phosphoproteome and the rapamycin-responsive phosphoproteome.</title>
<p>Distributions of the whole rat liver phosphoproteome are shown to the left for each analysis while the distributions of the phosphoproteins that were altered in response to rapamycin are shown on the right. The most abundant ten GO categories for each analysis are presented. Except where noted, an asterisk denotes a category that was significantly over-represented among rapamycin-sensitive candidate phosphoproteins as determined by chi-square analysis. Each category is followed by the number of genes identified in that category and the percentage of the total number of genes that were categorized for that analysis. For the cellular localization analysis, nine categories are presented for the rapamycin-responsive dataset since the same percentage values existed for the tenth and for all remaining categories. The asterisk denoting a whole phosphoproteome category indicates one that was significantly over-represented in that dataset. For the KEGG pathway categories, nine categories instead of ten are presented for the rapamycin-responsive dataset because the same percentage values were obtained for the tenth and multiple other categories.</p>
</caption>
<graphic xlink:href="pone.0021729.g005"></graphic>
</fig>
<p>KEGG Pathway analysis demonstrated that the phosphorylated proteins in the whole proteomic analysis were involved in 200 different pathways, including the insulin signaling pathway, pathways in cancer, and the mitogen-activated protein kinase (MAPK) signaling pathway. However, the set of rapamycin-sensitive proteins showed a significant over-representation of mTOR pathway constituents, which was the second most highly represented category after the insulin signaling pathway (<xref ref-type="fig" rid="pone-0021729-g005">Fig. 5</xref>, <italic>bottom panel</italic>). Based on the KEGG pathway analysis, the constituents of the mTOR pathway that were affected by rapamycin included raptor, ribosomal protein S6, AMP kinase 2, and the eukaryotic initiation factor eIF4b. Also included was p90S6 kinase 1, a component of the Erk signaling pathway that can be considered as involved in mTOR signaling based on cross-talk between pathways <xref ref-type="bibr" rid="pone.0021729-Keranen1">[51]</xref>.</p>
<p>To further characterize candidate phosphorylation sites that were altered in response to rapamycin, we used Kinexus' PhosphoNET kinase predictor software (<xref ref-type="table" rid="pone-0021729-t003">Table 3</xref>). In the human homologues of rat proteins, corresponding conserved phosphosites were searched. Sites on AMPK2 (Ser377), PRAS40 (Ser213), protein kinase C beta (Ser660) and raptor (Ser863), all of which were down-regulated in association with rapamycin administration, were strongly predicted to be targets of mTOR kinase. eIF3a (Ser584), hepatoma-derived growth factor 3 (Ser450), and PTPRF interacting protein (liprin) alpha 1 (Ser667) were predicted to be phosphorylated by p70S6 kinase. Two members of the phosphoinositide-3-kinase-related kinase family (of which mTOR is a member), ATM kinase and ATR kinase, were predicted to phosphorylate Bcl2-associated athanogene 3 (Ser174), Maf1 (Ser75) and peroxisome biogenesis factor 1 (Ser1181).</p>
<table-wrap id="pone-0021729-t003" position="float">
<object-id pub-id-type="doi">10.1371/journal.pone.0021729.t003</object-id>
<label>Table 3</label>
<caption>
<title>Kinase substrate predictions.</title>
</caption>
<alternatives>
<graphic id="pone-0021729-t003-3" xlink:href="pone.0021729.t003"></graphic>
<table frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
</colgroup>
<thead>
<tr>
<td align="left" colspan="1" rowspan="1">Protein name</td><td align="left" colspan="1" rowspan="1">Gene Symbol</td><td align="left" colspan="1" rowspan="1">P-site</td><td align="left" colspan="1" rowspan="1">Predicted Kinase 1<xref ref-type="table-fn" rid="nt101">*</xref>
</td><td align="left" colspan="1" rowspan="1">Predicted Kinase 2</td><td align="left" colspan="1" rowspan="1">Predicted Kinase 3</td><td align="left" colspan="1" rowspan="1">Predicted Kinase 4</td><td align="left" colspan="1" rowspan="1">Predicted Kinase 5</td>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">AKT1 substrate 1 (PRAS40) isoform CRA_d</td><td align="left" colspan="1" rowspan="1">AKT1S1</td><td align="left" colspan="1" rowspan="1">S203</td><td align="left" colspan="1" rowspan="1">CK2a1 (123)</td><td align="left" colspan="1" rowspan="1">CK2a2 (117)</td><td align="left" colspan="1" rowspan="1">p70S6K (106)</td><td align="left" colspan="1" rowspan="1">p70S6K (105)</td><td align="left" colspan="1" rowspan="1">SRPK1 (97)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AKT1 substrate 1 (PRAS40) isoform CRA_d</td><td align="left" colspan="1" rowspan="1">AKT1S1</td><td align="left" colspan="1" rowspan="1">S213</td><td align="left" colspan="1" rowspan="1">mTOR (247)</td><td align="left" colspan="1" rowspan="1">PCTAIRE3 (220)</td><td align="left" colspan="1" rowspan="1">PCTAIRE2 (214)</td><td align="left" colspan="1" rowspan="1">PCTAIRE1 (202)</td><td align="left" colspan="1" rowspan="1">ERK1 (200)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AMP-activated protein kinase alpha-2</td><td align="left" colspan="1" rowspan="1">PRKAA2</td><td align="left" colspan="1" rowspan="1">S377</td><td align="left" colspan="1" rowspan="1">mTOR (221)</td><td align="left" colspan="1" rowspan="1">NLK (170)</td><td align="left" colspan="1" rowspan="1">CDK1 (163)</td><td align="left" colspan="1" rowspan="1">CDK3 (161)</td><td align="left" colspan="1" rowspan="1">PCTAIRE3 (161)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Bcl2-associated athanogene 3</td><td align="left" colspan="1" rowspan="1">BAG3</td><td align="left" colspan="1" rowspan="1">S174</td><td align="left" colspan="1" rowspan="1">ATR (296)</td><td align="left" colspan="1" rowspan="1">ATM (245)</td><td align="left" colspan="1" rowspan="1">DNAPK (161)</td><td align="left" colspan="1" rowspan="1">SRPK1 (106)</td><td align="left" colspan="1" rowspan="1">SRPK2 (98)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Carbomyl-phosphate synthase 2</td><td align="left" colspan="1" rowspan="1">CAD</td><td align="left" colspan="1" rowspan="1">S1859</td><td align="left" colspan="1" rowspan="1">ROCK2 (195)</td><td align="left" colspan="1" rowspan="1">PIM1 (179)</td><td align="left" colspan="1" rowspan="1">ROCK1 (177)</td><td align="left" colspan="1" rowspan="1">MRCKb (171)</td><td align="left" colspan="1" rowspan="1">DMPK1 (169)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eukaryotic translation initiation factor 3A (ZH12 protein)</td><td align="left" colspan="1" rowspan="1">EIF3A</td><td align="left" colspan="1" rowspan="1">S584</td><td align="left" colspan="1" rowspan="1">SGK2 (163)</td><td align="left" colspan="1" rowspan="1">PIM1 (163)</td><td align="left" colspan="1" rowspan="1">PKG1 (158)</td><td align="left" colspan="1" rowspan="1">AKT2 (156)</td><td align="left" colspan="1" rowspan="1">PIM3 (156)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Growth factor receptor binding protein 7</td><td align="left" colspan="1" rowspan="1">GRB7</td><td align="left" colspan="1" rowspan="1">S364</td><td align="left" colspan="1" rowspan="1">JNK1 (194)</td><td align="left" colspan="1" rowspan="1">JNK2 (194)</td><td align="left" colspan="1" rowspan="1">JNK3 (193)</td><td align="left" colspan="1" rowspan="1">mTOR ( 184)</td><td align="left" colspan="1" rowspan="1">MAPK14 (151)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Hepatoma-derived growth factor 3 (HDGF2)</td><td align="left" colspan="1" rowspan="1">HDGFRP2</td><td align="left" colspan="1" rowspan="1">S450</td><td align="left" colspan="1" rowspan="1">PIM1 (259)</td><td align="left" colspan="1" rowspan="1">PIM2 (256)</td><td align="left" colspan="1" rowspan="1">PIM3 (256)</td><td align="left" colspan="1" rowspan="1">p70S6K (249)</td><td align="left" colspan="1" rowspan="1">p70S6Kb (249)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MAF1 (Repressor of RNA polymerase III transcription)</td><td align="left" colspan="1" rowspan="1">MAF1</td><td align="left" colspan="1" rowspan="1">S75</td><td align="left" colspan="1" rowspan="1">ATR (309)</td><td align="left" colspan="1" rowspan="1">ATM (250)</td><td align="left" colspan="1" rowspan="1">DNAPK (135)</td><td align="left" colspan="1" rowspan="1">CK2a1 (61)</td><td align="left" colspan="1" rowspan="1">mTOR (55)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Peroxisome biogenesis factor 1</td><td align="left" colspan="1" rowspan="1">PEX1</td><td align="left" colspan="1" rowspan="1">S1181</td><td align="left" colspan="1" rowspan="1">ATR (343)</td><td align="left" colspan="1" rowspan="1">ATM (275)</td><td align="left" colspan="1" rowspan="1">PIM1 (172)</td><td align="left" colspan="1" rowspan="1">PIM3 (172)</td><td align="left" colspan="1" rowspan="1">PKG1 (169)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Protein kinase C beta II</td><td align="left" colspan="1" rowspan="1">PRKCB</td><td align="left" colspan="1" rowspan="1">S660</td><td align="left" colspan="1" rowspan="1">mTOR (136)</td><td align="left" colspan="1" rowspan="1">ILK (98)</td><td align="left" colspan="1" rowspan="1">MAPK12 (77)</td><td align="left" colspan="1" rowspan="1">ERK1 (69)</td><td align="left" colspan="1" rowspan="1">ERK2 (65)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PTPRF interacting protein (liprin) alpha 1</td><td align="left" colspan="1" rowspan="1">PPFIA1</td><td align="left" colspan="1" rowspan="1">S667</td><td align="left" colspan="1" rowspan="1">p70S6K (130)</td><td align="left" colspan="1" rowspan="1">p70S6Kb (130)</td><td align="left" colspan="1" rowspan="1">PIM2 (110)</td><td align="left" colspan="1" rowspan="1">PIM1 (109)</td><td align="left" colspan="1" rowspan="1">SGK (104)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Raptor</td><td align="left" colspan="1" rowspan="1">RPTR</td><td align="left" colspan="1" rowspan="1">S863</td><td align="left" colspan="1" rowspan="1">mTOR (293)</td><td align="left" colspan="1" rowspan="1">PCTAIRE3 (267)</td><td align="left" colspan="1" rowspan="1">PCTAIRE2 (263)</td><td align="left" colspan="1" rowspan="1">PCTAIRE1 (250)</td><td align="left" colspan="1" rowspan="1">CDK1 (232)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ribosomal protein S6</td><td align="left" colspan="1" rowspan="1">RPS6</td><td align="left" colspan="1" rowspan="1">S235</td><td align="left" colspan="1" rowspan="1">PKCa (256)</td><td align="left" colspan="1" rowspan="1">PKCb (244)</td><td align="left" colspan="1" rowspan="1">ROCK2 (234)</td><td align="left" colspan="1" rowspan="1">MRCKb (231)</td><td align="left" colspan="1" rowspan="1">MRCKa (230)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ribosomal protein S6</td><td align="left" colspan="1" rowspan="1">RPS6</td><td align="left" colspan="1" rowspan="1">S236</td><td align="left" colspan="1" rowspan="1">PIM1 (250)</td><td align="left" colspan="1" rowspan="1">p70S6K (249)</td><td align="left" colspan="1" rowspan="1">p70S6Kb (249)</td><td align="left" colspan="1" rowspan="1">PIM3 (247)</td><td align="left" colspan="1" rowspan="1">PIM2 (237)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ribosomal protein S6</td><td align="left" colspan="1" rowspan="1">RPS6</td><td align="left" colspan="1" rowspan="1">S240</td><td align="left" colspan="1" rowspan="1">SRPK1 (162)</td><td align="left" colspan="1" rowspan="1">SRPK2 (156)</td><td align="left" colspan="1" rowspan="1">MSSK1 (151)</td><td align="left" colspan="1" rowspan="1">MRCKb (117)</td><td align="left" colspan="1" rowspan="1">MRCKa (116)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Sugen Kinase 269 (hypothetical protein XP_236266)</td><td align="left" colspan="1" rowspan="1">RGD1312026</td><td align="left" colspan="1" rowspan="1">S281</td><td align="left" colspan="1" rowspan="1">JNK2 (225)</td><td align="left" colspan="1" rowspan="1">JNK3 (219)</td><td align="left" colspan="1" rowspan="1">JNK1 (217)</td><td align="left" colspan="1" rowspan="1">PFTK1 (215)</td><td align="left" colspan="1" rowspan="1">MAPK14 (203)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ubiquitin specific peptidase 15, isoform CRA_a</td><td align="left" colspan="1" rowspan="1">USP15</td><td align="left" colspan="1" rowspan="1">S242</td><td align="left" colspan="1" rowspan="1">mTOR (178)</td><td align="left" colspan="1" rowspan="1">ERK1 (155)</td><td align="left" colspan="1" rowspan="1">ERK2 (155)</td><td align="left" colspan="1" rowspan="1">PFTK1 (129)</td><td align="left" colspan="1" rowspan="1">ERK5 (126)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ubiquitin specific protease 24</td><td align="left" colspan="1" rowspan="1">USP24</td><td align="left" colspan="1" rowspan="1">S1138</td><td align="left" colspan="1" rowspan="1">ATR (44)</td><td align="left" colspan="1" rowspan="1">ATM (42)</td><td align="left" colspan="1" rowspan="1">GSK3B (38)</td><td align="left" colspan="1" rowspan="1">PKACa (33)</td><td align="left" colspan="1" rowspan="1">SgK085 (33)</td>
</tr>
</tbody>
</table>
</alternatives>
<table-wrap-foot>
<fn id="nt101">
<label></label>
<p>*Shown are the 5 human protein kinases that are most likely to phosphorylate each of the designated phosphosites. Kinase prediction scores are shown in parentheses. The higher the kinase prediction score, the better the prospect that a kinase will phosphorylate a given site.</p>
</fn>
</table-wrap-foot>
</table-wrap>
</sec>
</sec>
<sec id="s4">
<title>Discussion</title>
<p>In recent years, several groups have performed comprehensive tissue phosphoproteome analyses <xref ref-type="bibr" rid="pone.0021729-Jin1">[19]</xref>&ndash;<xref ref-type="bibr" rid="pone.0021729-Han2">[23]</xref>, <xref ref-type="bibr" rid="pone.0021729-Xia1">[58]</xref>&ndash;<xref ref-type="bibr" rid="pone.0021729-Liao1">[60]</xref>. The animal tissue most often used for phosphoproteome analysis has been liver <xref ref-type="bibr" rid="pone.0021729-Jin1">[19]</xref>&ndash;<xref ref-type="bibr" rid="pone.0021729-Han2">[23]</xref>. The first such study, performed by Jin et al. <xref ref-type="bibr" rid="pone.0021729-Jin1">[19]</xref>, utilized iron IMAC for the enrichment of phosphorylated peptides and conventional linear ion-trap mass spectrometry (LTQ) for the phosphopeptide analysis. These investigators identified 26 nonredundant phosphorylation sites. A subsequent study <xref ref-type="bibr" rid="pone.0021729-Moser1">[20]</xref> utilized high capacity iron IMAC and a higher mass accuracy MS Q-TOF instrument to identify 339 non-redundant phosphorylation sites from over 200 proteins. The analyses performed by Villen et al. in 2007 <xref ref-type="bibr" rid="pone.0021729-Villen1">[21]</xref> was a breakthrough study. These investigators identified 5,635 nonredundant phosphorylation sites from 2,149 proteins, signifying the first in-depth global analysis of phosphopeptides from liver. Notably, these investigators used more selective iron IMAC beads and MS instruments of higher mass accuracy (the LTQ-FT and the LTQ-Orbitrap) as compared to previous studies. However, it is likely that the SCX pre-fractionation of tryptic peptides was a critical factor accounting for the increase in phosphopeptide identification.</p>
<p>Lack of reproducibility has been an issue with large scale MS-based phosphoproteomic profiling. Moser and White <xref ref-type="bibr" rid="pone.0021729-Moser1">[20]</xref> tested the reproducibility of their methodology by performing three replicate analyses of the same rat liver homogenate. They observed that 56&ndash;63% (131 out of 207&ndash;234) of the peptides from each analysis were observed in all three analyses. Another issue is method of analysis. Alcolea et al. <xref ref-type="bibr" rid="pone.0021729-Alcolea1">[61]</xref> found that analyzing the same phosphopeptide enriched murine NIH/3T3 fibroblast lysate by two different LC-MS/MS platforms based on Q-TOF and LTQ-Orbitrap mass spectrometers led to identification of partially overlapping, but also distinct, phosphoproteome profiles. The Q-TOF based platform resulted in 1,485 non-redundant phosphopeptide identifications, whereas the LTQ-Orbitrap based platform identified 4,308 non-redundant phosphopeptides. Only 1,077 of the total population of phosphopeptides were detected by both platforms. Analyzing duplicate samples by LC-MS/MS on the LTQ-Orbitrap platform showed that &sim;70% of the identified phosphopeptides were identical. In a study comparing the peptides identified using two workflows, TiO<sub>2</sub>-SCX and SCX-TiO<sub>2</sub>, the overlap was 58 and 51% for the two methods, respectively <xref ref-type="bibr" rid="pone.0021729-Wu1">[62]</xref>. A similar overlap of 60% was observed, when they performed replicate LC-MS/MS analyses of the same TiO<sub>2</sub>-SCX sample by an LTQ-Orbitrap.</p>
<p>These previous studies profiling the liver phosphoproteome have not been aimed at characterizing signaling events in a physiological context. Accomplishing this required improved sensitivity and accuracy, and a demonstration of reproducibility. We found that peptide abundance affects reproducibility, but that reproducibility could be enhanced by the performance of technical replicates. There are several indications that our methods were sufficient to detect mTORC1-mediated protein phosphorylation. These included the identification of known mTORC1 targets, the significant enrichment for mTOR signaling pathway constituents as indicated by pathway analysis, and the kinase prediction results.</p>
<p>In their seminal studies, Hunter and Sefton reported the relative abundances of pSer, pThr, and pTyr to be 90%, 10%, and 0.05%, respectively <xref ref-type="bibr" rid="pone.0021729-Hunter1">[63]</xref>. The order of magnitude higher tyrosine phosphorylation frequency in our results may be a function of the tendency for pTyr-containing peptides to yield better quality MS/MS spectra with collision-induced dissociation fragmentation. In contrast, phosphoserine and phosphothreonine containing peptides may produce low scores due to fragmentation patterns dominated by neutral loss of phosphate <xref ref-type="bibr" rid="pone.0021729-Villen1">[21]</xref>. We are left to conclude that the actual proportion of protein phosphorylation accounted for by pTyr is between 0.05% and 2%, but likely toward the lower end of this range.</p>
<p>Two groups have recently reported large-scale, MS-based analysis of TORC1-dependent protein phosphorylation events <xref ref-type="bibr" rid="pone.0021729-Huber1">[35]</xref>, <xref ref-type="bibr" rid="pone.0021729-Chen1">[48]</xref>. Chen et al. <xref ref-type="bibr" rid="pone.0021729-Chen1">[48]</xref> used epidermal growth factor-induced HeLa cell cultures and employed the use of stable isotope labeling in cell culture (SILAC) to achieve quantitative analyses. They identified 250 rapamycin-sensitive phosphorylation sites from 161 cellular proteins. Their main finding was the identification of CDC25B (Ser375) as the key phosphorylation event in mediating rapamycin-induced oncogenic Akt activation <xref ref-type="bibr" rid="pone.0021729-Chen1">[48]</xref>. Huber et al. <xref ref-type="bibr" rid="pone.0021729-Huber1">[35]</xref> used Saccharomyces cerevisiae with various genetic backgrounds for label-free quantitative phosphoproteomic screens. This study reported 41 rapamycin-sensitive yeast proteins and revealed that rapamycin-regulated Sch9 (a homolog of mammalian kinases Akt and p70S6K1) is a central coordinator of protein synthesis.</p>
<p>To our knowledge, our work represents the first in-depth, global analysis of rapamycin-dependent phosphoproteomics performed on whole tissue samples. One noteworthy result is the identification of several rapamycin-sensitive candidates that are related to translation. eIF3a is the largest component of the eIF3 complex, which is required for several steps in the initiation of protein synthesis <xref ref-type="bibr" rid="pone.0021729-Mayeur1">[64]</xref>. The eIF3 complex interacts with p70S6K under conditions of nutrient depletion or starvation <xref ref-type="bibr" rid="pone.0021729-Holz1">[65]</xref>. We found that rapamycin administration was associated with a marked reduction in phosphorylation of a component of eIF3, eIF3a, at Ser584 in all three animal sets. Although the function of this site has not been defined, a search of the Minimotif Miner (<ext-link ext-link-type="uri" xlink:href="http://mnm.engr.uconn.edu/">http://mnm.engr.uconn.edu/</ext-link>) database reveals that the [KR]xRxx[ST] consensus motif is also present in ribosomal protein S6, a substrate for p70S6K. Our observation is consistent with the hypothesis that refeeding of rats after starvation causes the activation of mTORC1, leading to phosphorylation and release of p70S6K from the eIF3 complex.</p>
<p>We observed rapamycin-sensitive phosphorylation events involving several other proteins associated with translation. Ribonuclease UK114, also identified as &ldquo;translational inhibitor protein p14.5&rdquo;, has been shown in previous studies to be related to inhibition of cell proliferation <xref ref-type="bibr" rid="pone.0021729-Morishita1">[66]</xref>, <xref ref-type="bibr" rid="pone.0021729-Oka1">[67]</xref>. We found that rapamycin administration was associated with a &gt;5-fold increase in phosphorylation of this protein at Thr10 and Ser11 in all three paired analyses. These sites have not been previously identified or characterized.</p>
<p>We also identified several known direct interactors of mTOR as rapamycin-sensitive phosphoproteins. Among these were the mTORC1 component raptor on Ser863 <xref ref-type="bibr" rid="pone.0021729-Foster2">[49]</xref> and PRAS40, a novel mTOR binding partner (53). Rapamycin has been shown to decrease the association of PRAS40 with mTORC1 proteins (53), an event for which the mechanism has not been elucidated. The exact mechanism by which PRAS40 inhibits mTORC1 activity is not well understood. We found that rapamycin treatment was associated with reduced phosphorylation of rat PRAS40 at Ser203 and Ser213 in all three paired analyses. A recent phosphopeptide mapping study identified Ser183, Ser212 and Ser221 as mTOR-dependent phosphorylation sites in human PRAS40 <xref ref-type="bibr" rid="pone.0021729-Wang1">[53]</xref>. Ser212 of human PRAS40, which is homologous to Ser213 in rat PRAS40, was not identified as sensitive to rapamycin treatment by these investigators.</p>
<p>AMPK is a critical sensor of metabolic stress that can turn off biosynthetic pathways when cellular ATP/AMP ratios decline <xref ref-type="bibr" rid="pone.0021729-Kimura1">[68]</xref>. The two AMP kinase isoforms, which generally function in the same manner (53), can inhibit mTORC1 by phosphorylating and activating TSC2. We detected a rapamycin-associated reduction in phosphorylation of AMPK2 at Ser377 in all three paired analyses. The Kinexus' kinase predictor software indicated that mTOR is a candidate kinase for this particular site. Our results may indicate a feedback mechanism between mTOR and AMPK through this rapamycin-sensitive phosphorylation event.</p>
<p>Gephyrin is a microtubule-associated protein involved in membrane protein-cytoskeleton interactions that is purported to directly interact with mTOR in a manner that is required for rapamycin-sensitive signaling <xref ref-type="bibr" rid="pone.0021729-Sabatini1">[54]</xref>. The underlying mechanism has not been identified. We found that rapamycin administration was associated with a &gt;12-fold increase in phosphorylation of gephyrin at Ser200, a site that has not been assigned a biological function.</p>
<p>Chen and coworkers detected more transcription-related proteins (9.3%) than translation-related proteins (5.4%) among their rapamycin-sensitive proteins <xref ref-type="bibr" rid="pone.0021729-Chen1">[48]</xref>, just as we did. However, the only rapamycin-sensitive phosphorylation events common to our dataset and theirs were the well-characterized ribosomal protein S6 sites. An RNA binding protein, the La ribonucleoprotein (LARP), was identified as a rapamycin-sensitive candidate in both studies. Phosphorylation of human LARP at Ser849, which is homologous to Ser648 of rat LARP, was reduced upon rapamycin exposure in the study by Chen et al. <xref ref-type="bibr" rid="pone.0021729-Chen1">[48]</xref>, while we observed an increase in the phosphorylation of rat LARP at the Thr644 and Ser648 sites in association with rapamycin administration. A comparison of our data to those of Huber and coworkers <xref ref-type="bibr" rid="pone.0021729-Huber1">[35]</xref> shows that in both cases Maf1 and S6 were the only common rapamycin-sensitive proteins previously reported in the literature.</p>
<p>While our approach allowed us to identify what may be novel, physiologically relevant rapamycin-sensitive sites, our study has some important limitations. One is incomplete coverage of any given protein due to a suboptimal density of tryptic sites. For example, lysine and arginine content within eIF4G1 is very high, so digestion with trypsin results in very short peptide fragments. This likely accounted for our inability to identify peptides containing the rapamycin-sensitive Ser1108 phosphorylation site in eIF4G1 <xref ref-type="bibr" rid="pone.0021729-Gruppuso1">[69]</xref>. Other established rapamycin-sensitive phosphorylation sites include p70S6K (Thr389) and 4E-BP1 (Thr37) and (Thr46) <xref ref-type="bibr" rid="pone.0021729-Hall1">[70]</xref>. We do not have a definitive explanation for the absence of p70S6K Thr389 in our analyses, though low abundance of this phosphoprotein is likely. While we detected the Thr37 and Thr46 phosphorylation sites in 4E-BP1, their phosphorylation was not affected by rapamycin in either our study or another published study <xref ref-type="bibr" rid="pone.0021729-Chen1">[48]</xref>. This may be consistent with recent biochemical studies indicating a complex mechanism behind the effect of rapamycin on 4E-BP1 phosphorylation <xref ref-type="bibr" rid="pone.0021729-Chen1">[48]</xref>, <xref ref-type="bibr" rid="pone.0021729-Choo2">[71]</xref>, <xref ref-type="bibr" rid="pone.0021729-Dowling1">[72]</xref>.</p>
<p>There are other limitations inherent in our approach. We began with a preparation of proteins that were soluble under aqueous, non-detergent-containing conditions. Since the presence of detergents in samples complicates MS analysis <xref ref-type="bibr" rid="pone.0021729-Thingholm2">[73]</xref>, <xref ref-type="bibr" rid="pone.0021729-Wu2">[74]</xref>, extension of our methods to lipid-soluble proteins may be challenging. Very low abundance proteins or proteins with a low stoichiometry of phosphorylation would not have been detected in our MS analysis due to sensitivity limits of the LTQ-FTICR classic mass spectrometer and the suppression of ionization efficiency with the detection of phosphorylated peptides in the positive ion mode <xref ref-type="bibr" rid="pone.0021729-Lemeer1">[75]</xref>. These issues notwithstanding, we made another observation of potential physiological significance. That was the high frequency with which previously unidentified phosphorylation sites were up-regulated in association with rapamycin administration. It is, of course, the case that rapamycin-sensitive phosphoproteins may not be direct targets of mTOR kinase. The up-regulation of phosphorylation in response to rapamycin can be accounted for by a downstream kinase that is activated upon its own dephosphorylation. An example of such a mechanism is eukaryotic elongation factor 2 kinase (eEF2K), which is activated in response to its dephosphorylation <xref ref-type="bibr" rid="pone.0021729-Knebel1">[76]</xref>. Alternatively, such changes may reflect modulation of phosphatase activity.</p>
<p>Identification of potential rapamycin-sensitive phosphorylation sites is only the first step in characterizing the signaling events we have identified. The assignment of functional significance to newly identified sites will require a traditional biochemical approach. Nonetheless, through the application of a phosphoproteomics approach to an <italic>in vivo</italic> model of mTORC1 signaling we have identified 67 rapamycin-sensitive candidate phosphorylation events. Identification of known rapamycin-sensitive phosphorylation sites supported the reliability of our analysis. We identified a high number of novel rapamycin-sensitive candidate phosphorylation sites on proteins that are related to transcription, translation or cell growth, or are known to interact with mTOR kinase. While contributing to the understanding of mTOR action, our results indicate the potential utility of phosphoproteomic profiling of <italic>in vivo</italic> tissues in identifying new targets for drug therapies.</p>
</sec>
<sec id="s5" sec-type="supplementary-material">
<title>Supporting Information</title>
<supplementary-material content-type="local-data" id="pone.0021729.s001">
<label>Table S1</label>
<caption>
<p>
<bold>Table of peptide sequences, phosphorylation site assignment statistics (standard deviation and coefficient of variation), and quantitative analysis for all data described in this manuscript.</bold>
</p>
<p>(XLS)</p>
</caption>
<media mime-subtype="vnd.ms-excel" mimetype="application" xlink:href="pone.0021729.s001.xls">
<caption>
<p>Click here for additional data file.</p>
</caption>
</media>
</supplementary-material>
<supplementary-material content-type="local-data" id="pone.0021729.s002">
<label>Table S2</label>
<caption>
<p>
<bold>Calculation of the inflection point for the distribution of rapamycin-induced fold-change in peptide abundance.</bold>
</p>
<p>(XLS)</p>
</caption>
<media mime-subtype="vnd.ms-excel" mimetype="application" xlink:href="pone.0021729.s002.xls">
<caption>
<p>Click here for additional data file.</p>
</caption>
</media>
</supplementary-material>
</sec>
</body>
<back>
<ack>
<p>We thank Dr. Jennifer Sanders for many helpful discussions and for her assistance in the performance of the animal studies.</p>
</ack>
<fn-group>
<fn fn-type="conflict">
<p>
<bold>Competing Interests: </bold>The authors have declared that no competing interests exist.</p>
</fn>
<fn fn-type="financial-disclosure">
<p>
<bold>Funding: </bold>This work was supported by USPHS grants R01HD024455 (to P.A.G.), and 1R01AI083636, 2P20RR015578 and R21AI083908 (to A.R.S.). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>
</fn>
</fn-group>
<ref-list>
<title>References</title>
<ref id="pone.0021729-Wullschleger1">
<label>1</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Wullschleger</surname>
<given-names>S</given-names>
</name>
<name>
<surname>Loewith</surname>
<given-names>R</given-names>
</name>
<name>
<surname>Hall</surname>
<given-names>MN</given-names>
</name>
</person-group>             
<year>2006</year>             
<article-title>TOR signaling in growth and metabolism.</article-title>             
<source>Cell</source>             
<volume>124</volume>             
<fpage>471</fpage>             
<lpage>484</lpage>          
<pub-id pub-id-type="pmid">16469695</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Alessi1">
<label>2</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Alessi</surname>
<given-names>DR</given-names>
</name>
<name>
<surname>Pearce</surname>
<given-names>LR</given-names>
</name>
<name>
<surname>Garcia-Martinez</surname>
<given-names>JM</given-names>
</name>
</person-group>             
<year>2009</year>             
<article-title>New insights into mTOR signaling: mTORC2 and beyond.</article-title>             
<source>Sci Signal</source>             
<volume>2</volume>             
<fpage>e27</fpage>          
</element-citation>
</ref>
<ref id="pone.0021729-Dunlop1">
<label>3</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Dunlop</surname>
<given-names>EA</given-names>
</name>
<name>
<surname>Tee</surname>
<given-names>AR</given-names>
</name>
</person-group>             
<year>2009</year>             
<article-title>Mammalian target of rapamycin complex 1: signalling inputs, substrates and feedback mechanisms.</article-title>             
<source>Cell Signal</source>             
<volume>21</volume>             
<fpage>827</fpage>             
<lpage>835</lpage>          
<pub-id pub-id-type="pmid">19166929</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Guertin1">
<label>4</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Guertin</surname>
<given-names>DA</given-names>
</name>
<name>
<surname>Sabatini</surname>
<given-names>DM</given-names>
</name>
</person-group>             
<year>2009</year>             
<article-title>The pharmacology of mTOR inhibition.</article-title>             
<source>Sci Signal</source>             
<volume>2</volume>             
<fpage>e24</fpage>          
</element-citation>
</ref>
<ref id="pone.0021729-Abraham1">
<label>5</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Abraham</surname>
<given-names>RT</given-names>
</name>
</person-group>             
<year>2002</year>             
<article-title>Identification of TOR signaling complexes: more TORC for the cell growth engine.</article-title>             
<source>Cell</source>             
<volume>111</volume>             
<fpage>9</fpage>             
<lpage>12</lpage>          
<pub-id pub-id-type="pmid">12372295</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Hay1">
<label>6</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Hay</surname>
<given-names>N</given-names>
</name>
<name>
<surname>Sonenberg</surname>
<given-names>N</given-names>
</name>
</person-group>             
<year>2004</year>             
<article-title>Upstream and downstream of mTOR.</article-title>             
<source>Genes Dev</source>             
<volume>18</volume>             
<fpage>1926</fpage>             
<lpage>1945</lpage>          
<pub-id pub-id-type="pmid">15314020</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Foster1">
<label>7</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Foster</surname>
<given-names>KG</given-names>
</name>
<name>
<surname>Fingar</surname>
<given-names>DC</given-names>
</name>
</person-group>             
<year>2010</year>             
<article-title>Mammalian target of rapamycin (mTOR): conducting the cellular signaling symphony.</article-title>             
<source>J Biol Chem</source>             
<volume>285</volume>             
<fpage>14071</fpage>             
<lpage>14077</lpage>          
<pub-id pub-id-type="pmid">20231296</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Hidalgo1">
<label>8</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Hidalgo</surname>
<given-names>M</given-names>
</name>
<name>
<surname>Rowinsky</surname>
<given-names>EK</given-names>
</name>
</person-group>             
<year>2000</year>             
<article-title>The rapamycin-sensitive signal transduction pathway as a target for cancer therapy.</article-title>             
<source>Oncogene</source>             
<volume>19</volume>             
<fpage>6680</fpage>             
<lpage>6686</lpage>          
<pub-id pub-id-type="pmid">11426655</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Loewith1">
<label>9</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Loewith</surname>
<given-names>R</given-names>
</name>
<name>
<surname>Jacinto</surname>
<given-names>E</given-names>
</name>
<name>
<surname>Wullschleger</surname>
<given-names>S</given-names>
</name>
<name>
<surname>Lorberg</surname>
<given-names>A</given-names>
</name>
<name>
<surname>Crespo</surname>
<given-names>JL</given-names>
</name>
<etal></etal>
</person-group>             
<year>2002</year>             
<article-title>Two TOR complexes, only one of which is rapamycin sensitive, have distinct roles in cell growth control.</article-title>             
<source>Mol Cell</source>             
<volume>10</volume>             
<fpage>457</fpage>             
<lpage>468</lpage>          
<pub-id pub-id-type="pmid">12408816</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Wool1">
<label>10</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Wool</surname>
<given-names>IG</given-names>
</name>
</person-group>             
<year>1979</year>             
<article-title>The structure and function of eukaryotic ribosomes.</article-title>             
<source>Annu Rev Biochem</source>             
<volume>48</volume>             
<fpage>719</fpage>             
<lpage>754</lpage>          
<pub-id pub-id-type="pmid">382996</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Gingras1">
<label>11</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Gingras</surname>
<given-names>AC</given-names>
</name>
<name>
<surname>Raught</surname>
<given-names>B</given-names>
</name>
<name>
<surname>Sonenberg</surname>
<given-names>N</given-names>
</name>
</person-group>             
<year>2004</year>             
<article-title>mTOR signaling to translation.</article-title>             
<source>Curr Top Microbiol Immunol</source>             
<volume>279</volume>             
<fpage>169</fpage>             
<lpage>197</lpage>          
<pub-id pub-id-type="pmid">14560958</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Ma1">
<label>12</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Ma</surname>
<given-names>XM</given-names>
</name>
<name>
<surname>Blenis</surname>
<given-names>J</given-names>
</name>
</person-group>             
<year>2009</year>             
<article-title>Molecular mechanisms of mTOR-mediated translational control.</article-title>             
<source>Nat Rev Mol Cell Biol</source>             
<volume>10</volume>             
<fpage>307</fpage>             
<lpage>318</lpage>          
<pub-id pub-id-type="pmid">19339977</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Nojima1">
<label>13</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Nojima</surname>
<given-names>H</given-names>
</name>
<name>
<surname>Tokunaga</surname>
<given-names>C</given-names>
</name>
<name>
<surname>Eguchi</surname>
<given-names>S</given-names>
</name>
<name>
<surname>Oshiro</surname>
<given-names>N</given-names>
</name>
<name>
<surname>Hidayat</surname>
<given-names>S</given-names>
</name>
<etal></etal>
</person-group>             
<year>2003</year>             
<article-title>The mammalian target of rapamycin (mTOR) partner, raptor, binds the mTOR substrates p70 S6 kinase and 4E-BP1 through their TOR signaling (TOS) motif.</article-title>             
<source>J Biol Chem</source>             
<volume>278</volume>             
<fpage>15461</fpage>             
<lpage>15464</lpage>          
<pub-id pub-id-type="pmid">12604610</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Choo1">
<label>14</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Choo</surname>
<given-names>AY</given-names>
</name>
<name>
<surname>Blenis</surname>
<given-names>J</given-names>
</name>
</person-group>             
<year>2009</year>             
<article-title>Not all substrates are treated equally: implications for mTOR, rapamycin-resistance and cancer therapy.</article-title>             
<source>Cell Cycle</source>             
<volume>8</volume>             
<fpage>567</fpage>             
<lpage>572</lpage>          
<pub-id pub-id-type="pmid">19197153</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Efeyan1">
<label>15</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Efeyan</surname>
<given-names>A</given-names>
</name>
<name>
<surname>Sabatini</surname>
<given-names>DM</given-names>
</name>
</person-group>             
<year>2010</year>             
<article-title>mTOR and cancer: many loops in one pathway.</article-title>             
<source>Curr Opin Cell Biol</source>             
<volume>22</volume>             
<fpage>169</fpage>             
<lpage>176</lpage>          
<pub-id pub-id-type="pmid">19945836</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Polak1">
<label>16</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Polak</surname>
<given-names>P</given-names>
</name>
<name>
<surname>Hall</surname>
<given-names>MN</given-names>
</name>
</person-group>             
<year>2009</year>             
<article-title>mTOR and the control of whole body metabolism.</article-title>             
<source>Curr Opin Cell Biol</source>             
<volume>21</volume>             
<fpage>209</fpage>             
<lpage>218</lpage>          
<pub-id pub-id-type="pmid">19261457</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Anand1">
<label>17</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Anand</surname>
<given-names>P</given-names>
</name>
<name>
<surname>Gruppuso</surname>
<given-names>PA</given-names>
</name>
</person-group>             
<year>2005</year>             
<article-title>The regulation of hepatic protein synthesis during fasting in the rat.</article-title>             
<source>J Biol Chem</source>             
<volume>280</volume>             
<fpage>16427</fpage>             
<lpage>16436</lpage>          
<pub-id pub-id-type="pmid">15716276</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Anand2">
<label>18</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Anand</surname>
<given-names>P</given-names>
</name>
<name>
<surname>Gruppuso</surname>
<given-names>PA</given-names>
</name>
</person-group>             
<year>2006</year>             
<article-title>Rapamycin inhibits liver growth during refeeding in rats via control of ribosomal protein translation but not cap-dependent translation initiation.</article-title>             
<source>J Nutr</source>             
<volume>136</volume>             
<fpage>27</fpage>             
<lpage>33</lpage>          
<pub-id pub-id-type="pmid">16365054</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Jin1">
<label>19</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Jin</surname>
<given-names>WH</given-names>
</name>
<name>
<surname>Dai</surname>
<given-names>J</given-names>
</name>
<name>
<surname>Zhou</surname>
<given-names>H</given-names>
</name>
<name>
<surname>Xia</surname>
<given-names>QC</given-names>
</name>
<name>
<surname>Zou</surname>
<given-names>HF</given-names>
</name>
<etal></etal>
</person-group>             
<year>2004</year>             
<article-title>Phosphoproteome analysis of mouse liver using immobilized metal affinity purification and linear ion trap mass spectrometry.</article-title>             
<source>Rapid Commun Mass Spectrom</source>             
<volume>18</volume>             
<fpage>2169</fpage>             
<lpage>2176</lpage>          
<pub-id pub-id-type="pmid">15378723</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Moser1">
<label>20</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Moser</surname>
<given-names>K</given-names>
</name>
<name>
<surname>White</surname>
<given-names>FM</given-names>
</name>
</person-group>             
<year>2006</year>             
<article-title>Phosphoproteomic analysis of rat liver by high capacity IMAC and LC-MS/MS.</article-title>             
<source>J Proteome Res</source>             
<volume>5</volume>             
<fpage>98</fpage>             
<lpage>104</lpage>          
<pub-id pub-id-type="pmid">16396499</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Villen1">
<label>21</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Villen</surname>
<given-names>J</given-names>
</name>
<name>
<surname>Beausoleil</surname>
<given-names>SA</given-names>
</name>
<name>
<surname>Gerber</surname>
<given-names>SA</given-names>
</name>
<name>
<surname>Gygi</surname>
<given-names>SP</given-names>
</name>
</person-group>             
<year>2007</year>             
<article-title>Large-scale phosphorylation analysis of mouse liver.</article-title>             
<source>Proc Natl Acad Sci U S A</source>             
<volume>104</volume>             
<fpage>1488</fpage>             
<lpage>1493</lpage>          
<pub-id pub-id-type="pmid">17242355</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Han1">
<label>22</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Han</surname>
<given-names>G</given-names>
</name>
<name>
<surname>Ye</surname>
<given-names>M</given-names>
</name>
<name>
<surname>Zhou</surname>
<given-names>H</given-names>
</name>
<name>
<surname>Jiang</surname>
<given-names>X</given-names>
</name>
<name>
<surname>Feng</surname>
<given-names>S</given-names>
</name>
<etal></etal>
</person-group>             
<year>2008</year>             
<article-title>Large-scale phosphoproteome analysis of human liver tissue by enrichment and fractionation of phosphopeptides with strong anion exchange chromatography.</article-title>             
<source>Proteomics</source>             
<volume>8</volume>             
<fpage>1346</fpage>             
<lpage>1361</lpage>          
<pub-id pub-id-type="pmid">18318008</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Han2">
<label>23</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Han</surname>
<given-names>G</given-names>
</name>
<name>
<surname>Ye</surname>
<given-names>M</given-names>
</name>
<name>
<surname>Liu</surname>
<given-names>H</given-names>
</name>
<name>
<surname>Song</surname>
<given-names>C</given-names>
</name>
<name>
<surname>Sun</surname>
<given-names>D</given-names>
</name>
<etal></etal>
</person-group>             
<year>2010</year>             
<article-title>Phosphoproteome analysis of human liver tissue by long-gradient nanoflow LC coupled with multiple stage MS analysis.</article-title>             
<source>Electrophoresis</source>             
<volume>31</volume>             
<fpage>1080</fpage>             
<lpage>1089</lpage>          
<pub-id pub-id-type="pmid">20166139</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Salomon1">
<label>24</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Salomon</surname>
<given-names>AR</given-names>
</name>
<name>
<surname>Ficarro</surname>
<given-names>SB</given-names>
</name>
<name>
<surname>Brill</surname>
<given-names>LM</given-names>
</name>
<name>
<surname>Brinker</surname>
<given-names>A</given-names>
</name>
<name>
<surname>Phung</surname>
<given-names>QT</given-names>
</name>
<etal></etal>
</person-group>             
<year>2003</year>             
<article-title>Profiling of tyrosine phosphorylation pathways in human cells using mass spectrometry.</article-title>             
<source>Proc Natl Acad Sci U S A</source>             
<volume>100</volume>             
<fpage>443</fpage>             
<lpage>448</lpage>          
<pub-id pub-id-type="pmid">12522270</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Zhang1">
<label>25</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Zhang</surname>
<given-names>Y</given-names>
</name>
<name>
<surname>Wolf-Yadlin</surname>
<given-names>A</given-names>
</name>
<name>
<surname>Ross</surname>
<given-names>PL</given-names>
</name>
<name>
<surname>Pappin</surname>
<given-names>DJ</given-names>
</name>
<name>
<surname>Rush</surname>
<given-names>J</given-names>
</name>
<etal></etal>
</person-group>             
<year>2005</year>             
<article-title>Time-resolved mass spectrometry of tyrosine phosphorylation sites in the epidermal growth factor receptor signaling network reveals dynamic modules.</article-title>             
<source>Mol Cell Proteomics</source>             
<volume>4</volume>             
<fpage>1240</fpage>             
<lpage>1250</lpage>          
<pub-id pub-id-type="pmid">15951569</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Cao1">
<label>26</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Cao</surname>
<given-names>L</given-names>
</name>
<name>
<surname>Yu</surname>
<given-names>K</given-names>
</name>
<name>
<surname>Salomon</surname>
<given-names>AR</given-names>
</name>
</person-group>             
<year>2006</year>             
<article-title>Phosphoproteomic analysis of lymphocyte signaling.</article-title>             
<source>Adv Exp Med Biol</source>             
<volume>584</volume>             
<fpage>277</fpage>             
<lpage>288</lpage>          
<pub-id pub-id-type="pmid">16802613</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Gronborg1">
<label>27</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Gronborg</surname>
<given-names>M</given-names>
</name>
<name>
<surname>Kristiansen</surname>
<given-names>TZ</given-names>
</name>
<name>
<surname>Stensballe</surname>
<given-names>A</given-names>
</name>
<name>
<surname>Andersen</surname>
<given-names>JS</given-names>
</name>
<name>
<surname>Ohara</surname>
<given-names>O</given-names>
</name>
<etal></etal>
</person-group>             
<year>2002</year>             
<article-title>A mass spectrometry-based proteomic approach for identification of serine/threonine-phosphorylated proteins by enrichment with phospho-specific antibodies: identification of a novel protein, Frigg, as a protein kinase A substrate.</article-title>             
<source>Mol Cell Proteomics</source>             
<volume>1</volume>             
<fpage>517</fpage>             
<lpage>527</lpage>          
<pub-id pub-id-type="pmid">12239280</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Villen2">
<label>28</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Villen</surname>
<given-names>J</given-names>
</name>
<name>
<surname>Gygi</surname>
<given-names>SP</given-names>
</name>
</person-group>             
<year>2008</year>             
<article-title>The SCX/IMAC enrichment approach for global phosphorylation analysis by mass spectrometry.</article-title>             
<source>Nat Protoc</source>             
<volume>3</volume>             
<fpage>1630</fpage>             
<lpage>1638</lpage>          
<pub-id pub-id-type="pmid">18833199</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Kweon1">
<label>29</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Kweon</surname>
<given-names>HK</given-names>
</name>
<name>
<surname>Hakansson</surname>
<given-names>K</given-names>
</name>
</person-group>             
<year>2006</year>             
<article-title>Selective zirconium dioxide-based enrichment of phosphorylated peptides for mass spectrometric analysis.</article-title>             
<source>Anal Chem</source>             
<volume>78</volume>             
<fpage>1743</fpage>             
<lpage>1749</lpage>          
<pub-id pub-id-type="pmid">16536406</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Larsen1">
<label>30</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Larsen</surname>
<given-names>MR</given-names>
</name>
<name>
<surname>Thingholm</surname>
<given-names>TE</given-names>
</name>
<name>
<surname>Jensen</surname>
<given-names>ON</given-names>
</name>
<name>
<surname>Roepstorff</surname>
<given-names>P</given-names>
</name>
<name>
<surname>Jorgensen</surname>
<given-names>TJ</given-names>
</name>
</person-group>             
<year>2005</year>             
<article-title>Highly selective enrichment of phosphorylated peptides from peptide mixtures using titanium dioxide microcolumns.</article-title>             
<source>Mol Cell Proteomics</source>             
<volume>4</volume>             
<fpage>873</fpage>             
<lpage>886</lpage>          
<pub-id pub-id-type="pmid">15858219</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Wolschin1">
<label>31</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Wolschin</surname>
<given-names>F</given-names>
</name>
<name>
<surname>Wienkoop</surname>
<given-names>S</given-names>
</name>
<name>
<surname>Weckwerth</surname>
<given-names>W</given-names>
</name>
</person-group>             
<year>2005</year>             
<article-title>Enrichment of phosphorylated proteins and peptides from complex mixtures using metal oxide/hydroxide affinity chromatography (MOAC).</article-title>             
<source>Proteomics</source>             
<volume>5</volume>             
<fpage>4389</fpage>             
<lpage>4397</lpage>          
<pub-id pub-id-type="pmid">16222723</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Hubbard1">
<label>32</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Hubbard</surname>
<given-names>MJ</given-names>
</name>
<name>
<surname>Cohen</surname>
<given-names>P</given-names>
</name>
</person-group>             
<year>1993</year>             
<article-title>On target with a new mechanism for the regulation of protein phosphorylation.</article-title>             
<source>Trends Biochem Sci</source>             
<volume>18</volume>             
<fpage>172</fpage>             
<lpage>177</lpage>          
<pub-id pub-id-type="pmid">8392229</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Olsen1">
<label>33</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Olsen</surname>
<given-names>JV</given-names>
</name>
<name>
<surname>Blagoev</surname>
<given-names>B</given-names>
</name>
<name>
<surname>Gnad</surname>
<given-names>F</given-names>
</name>
<name>
<surname>Macek</surname>
<given-names>B</given-names>
</name>
<name>
<surname>Kumar</surname>
<given-names>C</given-names>
</name>
<etal></etal>
</person-group>             
<year>2006</year>             
<article-title>Global, in vivo, and site-specific phosphorylation dynamics in signaling networks.</article-title>             
<source>Cell</source>             
<volume>127</volume>             
<fpage>635</fpage>             
<lpage>648</lpage>          
<pub-id pub-id-type="pmid">17081983</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Thingholm1">
<label>34</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Thingholm</surname>
<given-names>TE</given-names>
</name>
<name>
<surname>Jorgensen</surname>
<given-names>TJ</given-names>
</name>
<name>
<surname>Jensen</surname>
<given-names>ON</given-names>
</name>
<name>
<surname>Larsen</surname>
<given-names>MR</given-names>
</name>
</person-group>             
<year>2006</year>             
<article-title>Highly selective enrichment of phosphorylated peptides using titanium dioxide.</article-title>             
<source>Nat Protoc</source>             
<volume>1</volume>             
<fpage>1929</fpage>             
<lpage>1935</lpage>          
<pub-id pub-id-type="pmid">17487178</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Huber1">
<label>35</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Huber</surname>
<given-names>A</given-names>
</name>
<name>
<surname>Bodenmiller</surname>
<given-names>B</given-names>
</name>
<name>
<surname>Uotila</surname>
<given-names>A</given-names>
</name>
<name>
<surname>Stahl</surname>
<given-names>M</given-names>
</name>
<name>
<surname>Wanka</surname>
<given-names>S</given-names>
</name>
<etal></etal>
</person-group>             
<year>2009</year>             
<article-title>Characterization of the rapamycin-sensitive phosphoproteome reveals that Sch9 is a central coordinator of protein synthesis.</article-title>             
<source>Genes Dev</source>             
<volume>23</volume>             
<fpage>1929</fpage>             
<lpage>1943</lpage>          
<pub-id pub-id-type="pmid">19684113</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Yu1">
<label>36</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Yu</surname>
<given-names>K</given-names>
</name>
<name>
<surname>Salomon</surname>
<given-names>AR</given-names>
</name>
</person-group>             
<year>2009</year>             
<article-title>PeptideDepot: flexible relational database for visual analysis of quantitative proteomic data and integration of existing protein information.</article-title>             
<source>Proteomics</source>             
<volume>9</volume>             
<fpage>5350</fpage>             
<lpage>5358</lpage>          
<pub-id pub-id-type="pmid">19834895</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Yu2">
<label>37</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Yu</surname>
<given-names>K</given-names>
</name>
<name>
<surname>Sabelli</surname>
<given-names>A</given-names>
</name>
<name>
<surname>DeKeukelaere</surname>
<given-names>L</given-names>
</name>
<name>
<surname>Park</surname>
<given-names>R</given-names>
</name>
<name>
<surname>Sindi</surname>
<given-names>S</given-names>
</name>
<etal></etal>
</person-group>             
<year>2009</year>             
<article-title>Integrated platform for manual and high-throughput statistical validation of tandem mass spectra.</article-title>             
<source>Proteomics</source>             
<volume>9</volume>             
<fpage>3115</fpage>             
<lpage>3125</lpage>          
<pub-id pub-id-type="pmid">19526561</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Yu3">
<label>38</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Yu</surname>
<given-names>K</given-names>
</name>
<name>
<surname>Salomon</surname>
<given-names>AR</given-names>
</name>
</person-group>             
<year>2010</year>             
<article-title>HTAPP: high-throughput autonomous proteomic pipeline.</article-title>             
<source>Proteomics</source>             
<volume>10</volume>             
<fpage>2113</fpage>             
<lpage>2122</lpage>          
<pub-id pub-id-type="pmid">20336676</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Cao2">
<label>39</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Cao</surname>
<given-names>L</given-names>
</name>
<name>
<surname>Yu</surname>
<given-names>K</given-names>
</name>
<name>
<surname>Banh</surname>
<given-names>C</given-names>
</name>
<name>
<surname>Nguyen</surname>
<given-names>V</given-names>
</name>
<name>
<surname>Ritz</surname>
<given-names>A</given-names>
</name>
<etal></etal>
</person-group>             
<year>2007</year>             
<article-title>Quantitative time-resolved phosphoproteomic analysis of mast cell signaling.</article-title>             
<source>J Immunol</source>             
<volume>179</volume>             
<fpage>5864</fpage>             
<lpage>5876</lpage>          
<pub-id pub-id-type="pmid">17947660</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Nguyen1">
<label>40</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Nguyen</surname>
<given-names>V</given-names>
</name>
<name>
<surname>Cao</surname>
<given-names>L</given-names>
</name>
<name>
<surname>Lin</surname>
<given-names>JT</given-names>
</name>
<name>
<surname>Hung</surname>
<given-names>N</given-names>
</name>
<name>
<surname>Ritz</surname>
<given-names>A</given-names>
</name>
<etal></etal>
</person-group>             
<year>2009</year>             
<article-title>A new approach for quantitative phosphoproteomic dissection of signaling pathways applied to T cell receptor activation.</article-title>             
<source>Mol Cell Proteomics</source>             
<volume>8</volume>             
<fpage>2418</fpage>             
<lpage>2431</lpage>          
<pub-id pub-id-type="pmid">19605366</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Ficarro1">
<label>41</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Ficarro</surname>
<given-names>SB</given-names>
</name>
<name>
<surname>Salomon</surname>
<given-names>AR</given-names>
</name>
<name>
<surname>Brill</surname>
<given-names>LM</given-names>
</name>
<name>
<surname>Mason</surname>
<given-names>DE</given-names>
</name>
<name>
<surname>Stettler-Gill</surname>
<given-names>M</given-names>
</name>
<etal></etal>
</person-group>             
<year>2005</year>             
<article-title>Automated immobilized metal affinity chromatography/nano-liquid chromatography/electrospray ionization mass spectrometry platform for profiling protein phosphorylation sites.</article-title>             
<source>Rapid Commun Mass Spectrom</source>             
<volume>19</volume>             
<fpage>57</fpage>             
<lpage>71</lpage>          
<pub-id pub-id-type="pmid">15570572</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Elias1">
<label>42</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Elias</surname>
<given-names>JE</given-names>
</name>
<name>
<surname>Gygi</surname>
<given-names>SP</given-names>
</name>
</person-group>             
<year>2010</year>             
<article-title>Target-decoy search strategy for mass spectrometry-based proteomics.</article-title>             
<source>Methods Mol Biol</source>             
<volume>604</volume>             
<fpage>55</fpage>             
<lpage>71</lpage>          
<pub-id pub-id-type="pmid">20013364</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Beausoleil1">
<label>43</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Beausoleil</surname>
<given-names>SA</given-names>
</name>
<name>
<surname>Villen</surname>
<given-names>J</given-names>
</name>
<name>
<surname>Gerber</surname>
<given-names>SA</given-names>
</name>
<name>
<surname>Rush</surname>
<given-names>J</given-names>
</name>
<name>
<surname>Gygi</surname>
<given-names>SP</given-names>
</name>
</person-group>             
<year>2006</year>             
<article-title>A probability-based approach for high-throughput protein phosphorylation analysis and site localization.</article-title>             
<source>Nat Biotechnol</source>             
<volume>24</volume>             
<fpage>1285</fpage>             
<lpage>1292</lpage>          
<pub-id pub-id-type="pmid">16964243</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Beausoleil2">
<label>44</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Beausoleil</surname>
<given-names>SA</given-names>
</name>
<name>
<surname>Jedrychowski</surname>
<given-names>M</given-names>
</name>
<name>
<surname>Schwartz</surname>
<given-names>D</given-names>
</name>
<name>
<surname>Elias</surname>
<given-names>JE</given-names>
</name>
<name>
<surname>Villen</surname>
<given-names>J</given-names>
</name>
<etal></etal>
</person-group>             
<year>2004</year>             
<article-title>Large-scale characterization of HeLa cell nuclear phosphoproteins.</article-title>             
<source>Proc Natl Acad Sci U S A</source>             
<volume>101</volume>             
<fpage>12130</fpage>             
<lpage>12135</lpage>          
<pub-id pub-id-type="pmid">15302935</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Sugiyama1">
<label>45</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Sugiyama</surname>
<given-names>N</given-names>
</name>
<name>
<surname>Masuda</surname>
<given-names>T</given-names>
</name>
<name>
<surname>Shinoda</surname>
<given-names>K</given-names>
</name>
<name>
<surname>Nakamura</surname>
<given-names>A</given-names>
</name>
<name>
<surname>Tomita</surname>
<given-names>M</given-names>
</name>
<etal></etal>
</person-group>             
<year>2007</year>             
<article-title>Phosphopeptide enrichment by aliphatic hydroxy acid-modified metal oxide chromatography for nano-LC-MS/MS in proteomics applications.</article-title>             
<source>Mol Cell Proteomics</source>             
<volume>6</volume>             
<fpage>1103</fpage>             
<lpage>1109</lpage>          
<pub-id pub-id-type="pmid">17322306</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Kyono1">
<label>46</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Kyono</surname>
<given-names>Y</given-names>
</name>
<name>
<surname>Sugiyama</surname>
<given-names>N</given-names>
</name>
<name>
<surname>Imami</surname>
<given-names>K</given-names>
</name>
<name>
<surname>Tomita</surname>
<given-names>M</given-names>
</name>
<name>
<surname>Ishihama</surname>
<given-names>Y</given-names>
</name>
</person-group>             
<year>2008</year>             
<article-title>Successive and selective release of phosphorylated peptides captured by hydroxy acid-modified metal oxide chromatography.</article-title>             
<source>J Proteome Res</source>             
<volume>7</volume>             
<fpage>4585</fpage>             
<lpage>4593</lpage>          
<pub-id pub-id-type="pmid">18767875</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Bronshtein1">
<label>47</label>
<element-citation publication-type="book">             
<person-group person-group-type="author">
<name>
<surname>Bronshtein</surname>
<given-names>IN</given-names>
</name>
<name>
<surname>Semendyayev</surname>
<given-names>KA</given-names>
</name>
<name>
<surname>Musoil</surname>
<given-names>G</given-names>
</name>
<name>
<surname>Muehlig</surname>
<given-names>H</given-names>
</name>
</person-group>             
<year>2004</year>             
<source>Handbook of Mathematics</source>             
<publisher-loc>New York</publisher-loc>             
<publisher-name>Springer-Verlag.</publisher-name> 
<size units="page"></size>           
</element-citation>
</ref>
<ref id="pone.0021729-Chen1">
<label>48</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Chen</surname>
<given-names>RQ</given-names>
</name>
<name>
<surname>Yang</surname>
<given-names>QK</given-names>
</name>
<name>
<surname>Lu</surname>
<given-names>BW</given-names>
</name>
<name>
<surname>Yi</surname>
<given-names>W</given-names>
</name>
<name>
<surname>Cantin</surname>
<given-names>G</given-names>
</name>
<etal></etal>
</person-group>             
<year>2009</year>             
<article-title>CDC25B mediates rapamycin-induced oncogenic responses in cancer cells.</article-title>             
<source>Cancer Res</source>             
<volume>69</volume>             
<fpage>2663</fpage>             
<lpage>2668</lpage>          
<pub-id pub-id-type="pmid">19276368</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Foster2">
<label>49</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Foster</surname>
<given-names>KG</given-names>
</name>
<name>
<surname>Acosta-Jaquez</surname>
<given-names>HA</given-names>
</name>
<name>
<surname>Romeo</surname>
<given-names>Y</given-names>
</name>
<name>
<surname>Ekim</surname>
<given-names>B</given-names>
</name>
<name>
<surname>Soliman</surname>
<given-names>GA</given-names>
</name>
<etal></etal>
</person-group>             
<year>2010</year>             
<article-title>Regulation of mTOR complex 1 (mTORC1) by raptor Ser863 and multisite phosphorylation.</article-title>             
<source>J Biol Chem</source>             
<volume>285</volume>             
<fpage>80</fpage>             
<lpage>94</lpage>          
<pub-id pub-id-type="pmid">19864431</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Shor1">
<label>50</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Shor</surname>
<given-names>B</given-names>
</name>
<name>
<surname>Wu</surname>
<given-names>J</given-names>
</name>
<name>
<surname>Shakey</surname>
<given-names>Q</given-names>
</name>
<name>
<surname>Toral-Barza</surname>
<given-names>L</given-names>
</name>
<name>
<surname>Shi</surname>
<given-names>C</given-names>
</name>
<etal></etal>
</person-group>             
<year>2010</year>             
<article-title>Requirement of the mTOR kinase for the regulation of Maf1 phosphorylation and control of RNA polymerase III-dependent transcription in cancer cells.</article-title>             
<source>J Biol Chem</source>             
<volume>285</volume>             
<fpage>15380</fpage>             
<lpage>15392</lpage>          
<pub-id pub-id-type="pmid">20233713</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Keranen1">
<label>51</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Keranen</surname>
<given-names>LM</given-names>
</name>
<name>
<surname>Dutil</surname>
<given-names>EM</given-names>
</name>
<name>
<surname>Newton</surname>
<given-names>AC</given-names>
</name>
</person-group>             
<year>1995</year>             
<article-title>Protein kinase C is regulated in vivo by three functionally distinct phosphorylations.</article-title>             
<source>Curr Biol</source>             
<volume>5</volume>             
<fpage>1394</fpage>             
<lpage>1403</lpage>          
<pub-id pub-id-type="pmid">8749392</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Dalby1">
<label>52</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Dalby</surname>
<given-names>KN</given-names>
</name>
<name>
<surname>Morrice</surname>
<given-names>N</given-names>
</name>
<name>
<surname>Caudwell</surname>
<given-names>FB</given-names>
</name>
<name>
<surname>Avruch</surname>
<given-names>J</given-names>
</name>
<name>
<surname>Cohen</surname>
<given-names>P</given-names>
</name>
</person-group>             
<year>1998</year>             
<article-title>Identification of regulatory phosphorylation sites in mitogen-activated protein kinase (MAPK)-activated protein kinase-1a/p90rsk that are inducible by MAPK.</article-title>             
<source>J Biol Chem</source>             
<volume>273</volume>             
<fpage>1496</fpage>             
<lpage>1505</lpage>          
<pub-id pub-id-type="pmid">9430688</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Wang1">
<label>53</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Wang</surname>
<given-names>L</given-names>
</name>
<name>
<surname>Harris</surname>
<given-names>TE</given-names>
</name>
<name>
<surname>Lawrence</surname>
<given-names>JC</given-names>
</name>
</person-group>             
<year>2008</year>             
<article-title>Regulation of proline-rich Akt substrate of 40 kDa (PRAS40) function by mammalian target of rapamycin complex 1 (mTORC1)-mediated phosphorylation.</article-title>             
<source>J Biol Chem</source>             
<volume>283</volume>             
<fpage>15619</fpage>             
<lpage>15627</lpage>          
<pub-id pub-id-type="pmid">18372248</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Sabatini1">
<label>54</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Sabatini</surname>
<given-names>DM</given-names>
</name>
<name>
<surname>Barrow</surname>
<given-names>RK</given-names>
</name>
<name>
<surname>Blackshaw</surname>
<given-names>S</given-names>
</name>
<name>
<surname>Burnett</surname>
<given-names>PE</given-names>
</name>
<name>
<surname>Lai</surname>
<given-names>MM</given-names>
</name>
<etal></etal>
</person-group>             
<year>1999</year>             
<article-title>Interaction of RAFT1 with gephyrin required for rapamycin-sensitive signaling.</article-title>             
<source>Science</source>             
<volume>284</volume>             
<fpage>1161</fpage>             
<lpage>1164</lpage>          
<pub-id pub-id-type="pmid">10325225</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Cheng1">
<label>55</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Cheng</surname>
<given-names>SW</given-names>
</name>
<name>
<surname>Fryer</surname>
<given-names>LG</given-names>
</name>
<name>
<surname>Carling</surname>
<given-names>D</given-names>
</name>
<name>
<surname>Shepherd</surname>
<given-names>PR</given-names>
</name>
</person-group>             
<year>2004</year>             
<article-title>Thr2446 is a novel mammalian target of rapamycin (mTOR) phosphorylation site regulated by nutrient status.</article-title>             
<source>J Biol Chem</source>             
<volume>279</volume>             
<fpage>15719</fpage>             
<lpage>15722</lpage>          
<pub-id pub-id-type="pmid">14970221</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Vander1">
<label>56</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Vander</surname>
<given-names>HE</given-names>
</name>
<name>
<surname>Lee</surname>
<given-names>SI</given-names>
</name>
<name>
<surname>Bandhakavi</surname>
<given-names>S</given-names>
</name>
<name>
<surname>Griffin</surname>
<given-names>TJ</given-names>
</name>
<name>
<surname>Kim</surname>
<given-names>DH</given-names>
</name>
</person-group>             
<year>2007</year>             
<article-title>Insulin signalling to mTOR mediated by the Akt/PKB substrate PRAS40.</article-title>             
<source>Nat Cell Biol</source>             
<volume>9</volume>             
<fpage>316</fpage>             
<lpage>323</lpage>          
<pub-id pub-id-type="pmid">17277771</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Prasad1">
<label>57</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Prasad</surname>
<given-names>TS</given-names>
</name>
<name>
<surname>Kandasamy</surname>
<given-names>K</given-names>
</name>
<name>
<surname>Pandey</surname>
<given-names>A</given-names>
</name>
</person-group>             
<year>2009</year>             
<article-title>Human Protein Reference Database and Human Proteinpedia as discovery tools for systems biology.</article-title>             
<source>Methods Mol Biol</source>             
<volume>577</volume>             
<fpage>67</fpage>             
<lpage>79</lpage>          
<pub-id pub-id-type="pmid">19718509</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Xia1">
<label>58</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Xia</surname>
<given-names>Q</given-names>
</name>
<name>
<surname>Cheng</surname>
<given-names>D</given-names>
</name>
<name>
<surname>Duong</surname>
<given-names>DM</given-names>
</name>
<name>
<surname>Gearing</surname>
<given-names>M</given-names>
</name>
<name>
<surname>Lah</surname>
<given-names>JJ</given-names>
</name>
<etal></etal>
</person-group>             
<year>2008</year>             
<article-title>Phosphoproteomic analysis of human brain by calcium phosphate precipitation and mass spectrometry.</article-title>             
<source>J Proteome Res</source>             
<volume>7</volume>             
<fpage>2845</fpage>             
<lpage>2851</lpage>          
<pub-id pub-id-type="pmid">18510355</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Zanivan1">
<label>59</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Zanivan</surname>
<given-names>S</given-names>
</name>
<name>
<surname>Gnad</surname>
<given-names>F</given-names>
</name>
<name>
<surname>Wickstrom</surname>
<given-names>SA</given-names>
</name>
<name>
<surname>Geiger</surname>
<given-names>T</given-names>
</name>
<name>
<surname>Macek</surname>
<given-names>B</given-names>
</name>
<etal></etal>
</person-group>             
<year>2008</year>             
<article-title>Solid tumor proteome and phosphoproteome analysis by high resolution mass spectrometry.</article-title>             
<source>J Proteome Res</source>             
<volume>7</volume>             
<fpage>5314</fpage>             
<lpage>5326</lpage>          
<pub-id pub-id-type="pmid">19367708</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Liao1">
<label>60</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Liao</surname>
<given-names>L</given-names>
</name>
<name>
<surname>McClatchy</surname>
<given-names>DB</given-names>
</name>
<name>
<surname>Park</surname>
<given-names>SK</given-names>
</name>
<name>
<surname>Xu</surname>
<given-names>T</given-names>
</name>
<name>
<surname>Lu</surname>
<given-names>B</given-names>
</name>
<etal></etal>
</person-group>             
<year>2008</year>             
<article-title>Quantitative analysis of brain nuclear phosphoproteins identifies developmentally regulated phosphorylation events.</article-title>             
<source>J Proteome Res</source>             
<volume>7</volume>             
<fpage>4743</fpage>             
<lpage>4755</lpage>          
<pub-id pub-id-type="pmid">18823140</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Alcolea1">
<label>61</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Alcolea</surname>
<given-names>MP</given-names>
</name>
<name>
<surname>Kleiner</surname>
<given-names>O</given-names>
</name>
<name>
<surname>Cutillas</surname>
<given-names>PR</given-names>
</name>
</person-group>             
<year>2009</year>             
<article-title>Increased confidence in large-scale phosphoproteomics data by complementary mass spectrometric techniques and matching of phosphopeptide data sets.</article-title>             
<source>J Proteome Res</source>             
<volume>8</volume>             
<fpage>3808</fpage>             
<lpage>3815</lpage>          
<pub-id pub-id-type="pmid">19537829</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Wu1">
<label>62</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Wu</surname>
<given-names>J</given-names>
</name>
<name>
<surname>Warren</surname>
<given-names>P</given-names>
</name>
<name>
<surname>Shakey</surname>
<given-names>Q</given-names>
</name>
<name>
<surname>Sousa</surname>
<given-names>E</given-names>
</name>
<name>
<surname>Hill</surname>
<given-names>A</given-names>
</name>
<etal></etal>
</person-group>             
<year>2010</year>             
<article-title>Integrating titania enrichment, iTRAQ labeling, and Orbitrap CID-HCD for global identification and quantitative analysis of phosphopeptides.</article-title>             
<source>Proteomics</source>             
<volume>10</volume>             
<fpage>2224</fpage>             
<lpage>2234</lpage>          
<pub-id pub-id-type="pmid">20340162</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Hunter1">
<label>63</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Hunter</surname>
<given-names>T</given-names>
</name>
<name>
<surname>Sefton</surname>
<given-names>BM</given-names>
</name>
</person-group>             
<year>1980</year>             
<article-title>Transforming gene product of Rous sarcoma virus phosphorylates tyrosine.</article-title>             
<source>Proc Natl Acad Sci U S A</source>             
<volume>77</volume>             
<fpage>1311</fpage>             
<lpage>1315</lpage>          
<pub-id pub-id-type="pmid">6246487</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Mayeur1">
<label>64</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Mayeur</surname>
<given-names>GL</given-names>
</name>
<name>
<surname>Fraser</surname>
<given-names>CS</given-names>
</name>
<name>
<surname>Peiretti</surname>
<given-names>F</given-names>
</name>
<name>
<surname>Block</surname>
<given-names>KL</given-names>
</name>
<name>
<surname>Hershey</surname>
<given-names>JW</given-names>
</name>
</person-group>             
<year>2003</year>             
<article-title>Characterization of eIF3k: a newly discovered subunit of mammalian translation initiation factor elF3.</article-title>             
<source>Eur J Biochem</source>             
<volume>270</volume>             
<fpage>4133</fpage>             
<lpage>4139</lpage>          
<pub-id pub-id-type="pmid">14519125</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Holz1">
<label>65</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Holz</surname>
<given-names>MK</given-names>
</name>
<name>
<surname>Ballif</surname>
<given-names>BA</given-names>
</name>
<name>
<surname>Gygi</surname>
<given-names>SP</given-names>
</name>
<name>
<surname>Blenis</surname>
<given-names>J</given-names>
</name>
</person-group>             
<year>2005</year>             
<article-title>mTOR and S6K1 mediate assembly of the translation preinitiation complex through dynamic protein interchange and ordered phosphorylation events.</article-title>             
<source>Cell</source>             
<volume>123</volume>             
<fpage>569</fpage>             
<lpage>580</lpage>          
<pub-id pub-id-type="pmid">16286006</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Morishita1">
<label>66</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Morishita</surname>
<given-names>R</given-names>
</name>
<name>
<surname>Kawagoshi</surname>
<given-names>A</given-names>
</name>
<name>
<surname>Sawasaki</surname>
<given-names>T</given-names>
</name>
<name>
<surname>Madin</surname>
<given-names>K</given-names>
</name>
<name>
<surname>Ogasawara</surname>
<given-names>T</given-names>
</name>
<etal></etal>
</person-group>             
<year>1999</year>             
<article-title>Ribonuclease activity of rat liver perchloric acid-soluble protein, a potent inhibitor of protein synthesis.</article-title>             
<source>J Biol Chem</source>             
<volume>274</volume>             
<fpage>20688</fpage>             
<lpage>20692</lpage>          
<pub-id pub-id-type="pmid">10400702</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Oka1">
<label>67</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Oka</surname>
<given-names>T</given-names>
</name>
<name>
<surname>Tsuji</surname>
<given-names>H</given-names>
</name>
<name>
<surname>Noda</surname>
<given-names>C</given-names>
</name>
<name>
<surname>Sakai</surname>
<given-names>K</given-names>
</name>
<name>
<surname>Hong</surname>
<given-names>YM</given-names>
</name>
<etal></etal>
</person-group>             
<year>1995</year>             
<article-title>Isolation and characterization of a novel perchloric acid-soluble protein inhibiting cell-free protein synthesis.</article-title>             
<source>J Biol Chem</source>             
<volume>270</volume>             
<fpage>30060</fpage>             
<lpage>30067</lpage>          
<pub-id pub-id-type="pmid">8530410</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Kimura1">
<label>68</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Kimura</surname>
<given-names>N</given-names>
</name>
<name>
<surname>Tokunaga</surname>
<given-names>C</given-names>
</name>
<name>
<surname>Dalal</surname>
<given-names>S</given-names>
</name>
<name>
<surname>Richardson</surname>
<given-names>C</given-names>
</name>
<name>
<surname>Yoshino</surname>
<given-names>K</given-names>
</name>
<etal></etal>
</person-group>             
<year>2003</year>             
<article-title>A possible linkage between AMP-activated protein kinase (AMPK) and mammalian target of rapamycin (mTOR) signalling pathway.</article-title>             
<source>Genes Cells</source>             
<volume>8</volume>             
<fpage>65</fpage>             
<lpage>79</lpage>          
<pub-id pub-id-type="pmid">12558800</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Gruppuso1">
<label>69</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Gruppuso</surname>
<given-names>PA</given-names>
</name>
<name>
<surname>Tsai</surname>
<given-names>SW</given-names>
</name>
<name>
<surname>Boylan</surname>
<given-names>JM</given-names>
</name>
<name>
<surname>Sanders</surname>
<given-names>JA</given-names>
</name>
</person-group>             
<year>2008</year>             
<article-title>Hepatic translation control in the late-gestation fetal rat.</article-title>             
<source>Am J Physiol Regul Integr Comp Physiol</source>             
<volume>295</volume>             
<fpage>R558</fpage>             
<lpage>R567</lpage>          
<pub-id pub-id-type="pmid">18565838</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Hall1">
<label>70</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Hall</surname>
<given-names>MN</given-names>
</name>
</person-group>             
<year>2008</year>             
<article-title>mTOR-what does it do?</article-title>             
<source>Transplant Proc</source>             
<volume>40</volume>             
<fpage>S5</fpage>             
<lpage>S8</lpage>          
<pub-id pub-id-type="pmid">19100909</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Choo2">
<label>71</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Choo</surname>
<given-names>AY</given-names>
</name>
<name>
<surname>Yoon</surname>
<given-names>SO</given-names>
</name>
<name>
<surname>Kim</surname>
<given-names>SG</given-names>
</name>
<name>
<surname>Roux</surname>
<given-names>PP</given-names>
</name>
<name>
<surname>Blenis</surname>
<given-names>J</given-names>
</name>
</person-group>             
<year>2008</year>             
<article-title>Rapamycin differentially inhibits S6Ks and 4E-BP1 to mediate cell-type-specific repression of mRNA translation.</article-title>             
<source>Proc Natl Acad Sci U S A</source>             
<volume>105</volume>             
<fpage>17414</fpage>             
<lpage>17419</lpage>          
<pub-id pub-id-type="pmid">18955708</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Dowling1">
<label>72</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Dowling</surname>
<given-names>RJ</given-names>
</name>
<name>
<surname>Topisirovic</surname>
<given-names>I</given-names>
</name>
<name>
<surname>Alain</surname>
<given-names>T</given-names>
</name>
<name>
<surname>Bidinosti</surname>
<given-names>M</given-names>
</name>
<name>
<surname>Fonseca</surname>
<given-names>BD</given-names>
</name>
<etal></etal>
</person-group>             
<year>2010</year>             
<article-title>mTORC1-mediated cell proliferation, but not cell growth, controlled by the 4E-BPs.</article-title>             
<source>Science</source>             
<volume>328</volume>             
<fpage>1172</fpage>             
<lpage>1176</lpage>          
<pub-id pub-id-type="pmid">20508131</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Thingholm2">
<label>73</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Thingholm</surname>
<given-names>TE</given-names>
</name>
<name>
<surname>Jensen</surname>
<given-names>ON</given-names>
</name>
<name>
<surname>Larsen</surname>
<given-names>MR</given-names>
</name>
</person-group>             
<year>2009</year>             
<article-title>Analytical strategies for phosphoproteomics.</article-title>             
<source>Proteomics</source>             
<volume>9</volume>             
<fpage>1451</fpage>             
<lpage>1468</lpage>          
<pub-id pub-id-type="pmid">19235172</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Wu2">
<label>74</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Wu</surname>
<given-names>J</given-names>
</name>
<name>
<surname>Warren</surname>
<given-names>P</given-names>
</name>
<name>
<surname>Shakey</surname>
<given-names>Q</given-names>
</name>
<name>
<surname>Sousa</surname>
<given-names>E</given-names>
</name>
<name>
<surname>Hill</surname>
<given-names>A</given-names>
</name>
<etal></etal>
</person-group>             
<year>2010</year>             
<article-title>Integrating titania enrichment, iTRAQ labeling, and Orbitrap CID-HCD for global identification and quantitative analysis of phosphopeptides.</article-title>             
<source>Proteomics</source>             
<volume>10</volume>             
<fpage>2224</fpage>             
<lpage>2234</lpage>          
<pub-id pub-id-type="pmid">20340162</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Lemeer1">
<label>75</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Lemeer</surname>
<given-names>S</given-names>
</name>
<name>
<surname>Heck</surname>
<given-names>AJ</given-names>
</name>
</person-group>             
<year>2009</year>             
<article-title>The phosphoproteomics data explosion.</article-title>             
<source>Curr Opin Chem Biol</source>             
<volume>13</volume>             
<fpage>414</fpage>             
<lpage>420</lpage>          
<pub-id pub-id-type="pmid">19620020</pub-id>
</element-citation>
</ref>
<ref id="pone.0021729-Knebel1">
<label>76</label>
<element-citation publication-type="journal">             
<person-group person-group-type="author">
<name>
<surname>Knebel</surname>
<given-names>A</given-names>
</name>
<name>
<surname>Morrice</surname>
<given-names>N</given-names>
</name>
<name>
<surname>Cohen</surname>
<given-names>P</given-names>
</name>
</person-group>             
<year>2001</year>             
<article-title>A novel method to identify protein kinase substrates: eEF2 kinase is phosphorylated and inhibited by SAPK4/p38delta.</article-title>             
<source>EMBO J</source>             
<volume>20</volume>             
<fpage>4360</fpage>             
<lpage>4369</lpage>          
<pub-id pub-id-type="pmid">11500363</pub-id>
</element-citation>
</ref>
</ref-list>
</back>
</article>
